# Molecular Determinants of HIV-2 R5-X4 Tropism in the V3 Loop: Development of a New Genotypic Tool 

Benoit Visseaux, 1,4,a Margarita Hurtado-Nedelec, ,,a Charlotte Charpentier, 1,4 Gilles Collin, 1,4 Alexandre Storto, 1,4<br>Sophie Matheron, 3,4 Lucile Larrouy, 1,4 Florence Damond, 1,4 Françoise Brun-Vézinet, 1,4 Diane Descamps, ${ }^{1,4}$ and<br>the ANRS CO 05 HIV-2 Cohort<br>'Laboratoire de Virologie, 2Département d'Hématologie et Immunologie, and ${ }^{3}$ Service des Maladies Infectieuses et Tropicales, Assistance Publique-Hôpitaux de<br>Paris, Hôpital Bichat-Claude Bernard, and ${ }^{4} E A 4409$ Paris-Diderot University, France

Objective. The use of CCR5 inhibitors requires a tool to predict human immunodeficiency virus type 2 (HIV-2) tropism, as established in HIV-1. The aim of our study was to identify genotypic determinants of HIV-2 tropism located in the gp105 V3 loop.

Methods. HIV-2 tropism phenotypic assays were performed on 53 HIV-2 clinical isolates using GFP expressing human osteosarcoma T4 [GHOST(3)] cell lines expressing CD4 and CCR5 or CXCR4 coreceptors. The gp105 V3 loop was sequenced and analyzed.

Results. Thirty-four HIV-2 isolates were classified as R5, 7 as X4, and 12 as X4/R5 (dual). Substitution at residue 18 was always associated with a dual/ $\mathrm{X} 4$ tropism $(P<.00001)$. The following determinants were associated with dual/X4 tropism: a global net charge of more than $+6(P<.00001)$, V19K/R mutation $(P<.00001), \mathrm{S} 22 \mathrm{~A} / \mathrm{F} / \mathrm{Y}$ mutation $(P<.002)$, Q23R mutation $(P<.00001)$, and insertions at residue $24(P<.00001)$, $\mathrm{I} 25 \mathrm{~L} / \mathrm{Y}(P<.0004)$, R28K $(P<.0004)$, and R30K $(P<.014)$. These mutations were not found in R5 isolates, except R28K and R30K, which were detected in 4 and 5 R5 isolates, respectively. The 4 major genotypic determinants of dual/X4 tropism were mutation at residue 18, V19 K/R mutation, insertions at residue 24, and V3 global net charge.

Conclusions. We established a strong association between HIV-2 phenotypic tropism and V3-loop sequences, allowing for the prediction of R5- and/or X4-tropic viruses in HIV-2 infection.

The human immunodeficiency virus (HIV) is characterized by a high level of genetic diversity within HIV type 1 (HIV-1) and HIV type 2 (HIV-2). HIV-2 infections are mainly restricted to West Africa, including Guinea-Bissau, Gambia, Senegal, and Guinea. Some European countries are also concerned about HIV-2 infection, which represents 5\% of HIV infection in a series of patients in Portugal [1] and 2\% of the new HIV infections in France [2].

HIV-2 clinical isolates appear to be susceptible to nucleoside reverse-transcriptase inhibitors and to the new drug class of integrase inhibitors [3, 4]. However, they are naturally resistant to nonnucleoside reverse-transcriptase inhibitors $[5,6]$ and to fusion-inhibitor drug classes $[6,7]$. In addition, HIV-2 shows decreased susceptibility to some protease inhibitors, such as amprenavir, atazanavir, and tipranavir $[6,8]$. In this context, the new class of CCR5 coreceptor (R5) inhibitors with maraviroc may provide a new therapeutic opportunity in HIV-2 infection management. However, few data are available on maraviroc use in HIV-2 infection. One study found in vitro antiviral activity of nonlicensed CCR5 inhibitors on HIV-2 [9] and 2 HIV-2-infected patients with a multiresistant virus showed an early virological response after receiving a multiple-drug salvage regimen with maraviroc [10,11].

The process of HIV-2 entry involves identical steps to HIV-1, including similar virus-cell interactions, first between the HIV-2 gp105 envelope glycoprotein and CD4 receptor and then between the gp105 V3 loop and CCR5 or CXCR4 coreceptors [12, 13]. Furthermore, HIV-2 seems to be able to use in vitro a broad range of coreceptors, such as CCR1, CCR2b, CCR3, CCR8, BOB/GPR15, or BONZO/CXCR6 [12, 14-17]. However, discrepancies exist between studies and cell lines models for tropism determination. Indeed, U87 cell lines showed higher level of nonspecific infection than GFP expressing human osteosarcoma T4 [GHOST(3)] cell lines [17, 18]. In the GHOST(3) cell line model, only CCR5, GPR15, and CXCR6 were efficiently used in vitro by R5-tropic HIV-2 viruses, and only CXCR4 was efficiently used by X4-tropic HIV-2 virus [18].Moreover, 2 studies assessed in vitro the relative contribution of these different coreceptors in HIV-2 infection using coreceptorspecific inhibitors on peripheral-blood mononudear cells $\dot{(\text{PBMCs) or a PBMC model with CCR5}^{+/+}}$ and CCR5 $^-/-$ blood donors, showing that only CCR5- or CXCR4-dependent infection is really efficient on PBMC [13,19].

CCR5 inhibitors are ineffective against the CXCR4 (X4)-using virus $[20,21]$. Thus, it is mandatory to assess viral tropism in order to determine a patient's eligibility for maraviroc therapy. In HIV-1 infection, tropism can be determined by a phenotypic recombinant test (Trofile; Monogram Biosciences) [22] or by a genotypic test that analyzes the gp120 V3-loop sequences [23, 24]. Genotypic interpretation of V3-loop sequences for tropism determination can be performed using different algorithms $[25,26]$ that take into account the major genotypic determinants of coreceptors used in the HIV-1 genetic context [27, 28].

In HIV-2 infection, several studies have evaluated the correlations between the gp105 V3-loop sequence and HIV-2 tropism, as assessed by phenotypic tests [15-17, 29-31]. All these studies have reported a few HIV-2 strains, and have reached discordant conclusions. The aim of this study was to establish a genotypic tool for HIV-2-tropism determination, as a first step in studying tropism implications in HIV-2 infection.

## MATERIALS AND METHODS

## Study Patients and Virus Stocks

Fifty-three HIV-2 clinical isolates, from 53 HIV-2-infected, maraviroc-naive patients of the French Agence Nationale de Recherches sur le SIDA et les hépatites virales (ANRS) HIV-2 cohort (CO 05), were isolated by cocultivation of phytohemagglutininactivated PBMCs from a pool of healthy blood donors, as described elsewhere [3]. Written consent was provided by all patients at the time of inclusion in the French ANRS CO 05 HIV-2 cohort. Among the 53 patients, $37(70 \%)$ were infected with HIV-2 group A, 15 (28\%) with HIV-2 group B, and 1 with HIV-2 group H (2\%). The HIV-2 group A reference strain, HIV- $2_{\text {ROD }}$, using both CCR5 and CXCR4 coreceptors, was included in our series.

## Cell Lines

Peripheral-blood mononuclear cells were obtained from HIVseronegative blood donors. They were stimulated with phytohemagglutinin and cultured, as described elsewhere [3]. GHOST(3) cell lines are human osteosarcoma cells engineered to stably express the CD4 receptor and one HIV coreceptor, CCR5 or CXCR4. Both GHOST(3) cell lines (CD4 plus CCR5 or CD4 plus CXCR4) also carry a humanized green fluorescent protein (GFP) gene under the control of a HIV-2 long terminal repeat [15]. Infection with HIV and consecutive production of the Tat protein induces expression of GFP, which can be detected by fluorescent microscopy or cytometric analysis. GHOST(3) cell lines were cultured in Dulbecco's Modified Eagle Medium (DMEM) medium (Invitrogen) containing 10\% fetal bovine serum (FBS), hygromycin $(100 \mu \mathrm{g} / \mathrm{mL})$, geneticin $(500 \mu \mathrm{g} / \mathrm{mL})$, and puromycin $(1 \mu \mathrm{g} / \mathrm{mL})$. GHOST(3) High CCR5 coreceptors (Hi-5) cells and GHOST(3) CXCR4 cells were provided by the National Institutes of Health (NIH) AIDS Reagent Program. At the time of this work, parental cell lines expressing only the CD4 receptor without any coreceptor were not available and were still not available at the writing of this work.

## Phenotypic Tropism Assay

GHOST(3) cells were seeded in 24-well plates at 40000 per well in growth medium and incubated for 24 hours at $37^{\circ} \mathrm{C}$ in $5 \%$ carbon dioxide $\left(\mathrm{CO}_{2}\right)$. After incubation, the culture medium was removed from the wells, and $300 \mu \mathrm{L}$ of cell-free virus, at a titer between $10^{7}$ and $10^{8}$ copies/ $\mathrm{mL}$ and contained in DMEM, with $10 \%$ FBS and $7.5 \mu \mathrm{g} / \mathrm{L}$ polybrene, was added to the cells. Plates were immediately centrifuged for 30 minutes at $2300 \mathrm{~g}$. The centrifugation step and the addition of polybrene, an anionic polymer that enhances viruscell contacts by shielding charge and virus-aggregation mechanisms, were added to increase infection efficiency [32-34]. Plates were incubated for 4 hours at $37^{\circ} \mathrm{C}$ in $5 \% \mathrm{CO}_{2}$. The viral supernatant was then removed, and $1 \mathrm{~mL}$ of culture medium was added to each well. Plates were incubated for 3 days at $37^{\circ} \mathrm{C}$ in $5 \% \mathrm{CO}_{2}$. Cells were then observed for cytopathogenic effects, the culture medium was removed, and $90 \mu \mathrm{L}$ of trypsin was added. After 5 minutes of incubation at $37^{\circ} \mathrm{C}$, trypsin was inactivated by the addition of $300 \mu \mathrm{L}$ of phosphate-buffered saline and 20\% FBS. Cells were analyzed within an hour by flow cytometry, after staining with propidium iodide in order to assess cellular viability. A negative control, containing only culture medium, was included in each experiment. The ratio-to-cell-negative (RTCN) value was used to quantify cell infection $[18,35]$. The RTCN value is the ratio of the percentage of fluorescence-positive cells to their mean fluorescence intensity (FI), calculated as follows: $\mathrm{RTCN}=(\%$ of GFP-positive cells $\times$ FI) of infected cells/(\% of GFP-positive cells $\times$ FI) of uninfected cells. A RTCN value of 5 was taken as the negative cutoff threshold. In our series, when the proportion of GFP-positive cells was $<10 \%$, the phenotypic tropism assay was performed again using a 2 -fold larger viral inoculum. No discordance was found with these controls (data not shown).

GHOST(3) cell lines are known to endogenously express in low-level proportion the CXCR4 coreceptor [36, 37]. Thus, low levels of infection were described in previous study with HIV-2 X4 isolates in GHOST(3) cell lines [18]. In regard to these latter data, in our study all HIV-2 isolates with a CCR5 cell RTCN value $<15 \%$ of the CXCR 4 cell RTCN value were considered X4 tropic. A very low-level infection of parental GHOST cell lines has been described elsewhere in some HIV-2 isolates, independently of CCR5 or CXCR4 coreceptors [18]. The proposed mechanism could be coreceptor-independent infection or alternative coreceptor usage. However, these infections are infrequent and have a very low efficiency. In regards to the data obtained by Blaak et al [18], all HIV-2 isolates that have a CXCR4 RTCN value $<5 \%$ of the CCR5 cell RTCN value were considered R5 tropic.

## HIV-2 Gene Sequencing

HIV-2 RNA was extracted from $1 \mathrm{~mL}$ of viral supernatant using a Total NA Large Volume kit (Roche Diagnostic) on a MagnaPure automate (Roche Diagnostic), according to the manufacturer's recommendations. A 1000-base pair fragment of gp105, encompassing the V3 loop, was first amplified by reverse-transcription polymerase chain reaction (RT-PCR) according to the manufacturer's instructions (Titan One-Tube RT-PCR kit; Roche Diagnostic) using the following primers: 1S: $5^{\prime}$-ATTTTCCAGGTSTGGCARAGGTC-3' (nucleotide 5476-5498 according HIV- $2_{\text {ROD }}$ sequence) and 1AS: 5 '-GCACATCCCCATGAATTTAG-3' (nucleotide 7920-7939). The RT step was carried out at $50^{\circ} \mathrm{C}$ for 30 minutes, followed by $94^{\circ} \mathrm{C}$ for 2 minutes. The PCR cycling parameters were 30 repeat cycles $\left(94^{\circ} \mathrm{C}\right.$ for 30 seconds, $55^{\circ} \mathrm{C}$ for 30 seconds, and $68^{\circ} \mathrm{C}$ for 120 seconds) plus $68^{\circ} \mathrm{C}$ for 7 minutes. This was followed by a second round of PCR using the following primers: 2S: $5^{\prime}$-AGARTCATGTGAYAARCAYTATTGGGA-3' (nucleotide 6779-6805) and 2R: 5'-GCTGTTGCTGYTGCTGCACTATCC-3' (nucleotide 7783-7806).

Nested-PCR reactions were performed with Taq polymerase (Applied Biosystems) and $2.5 \mu \mathrm{L}$ of internal $2 \mathrm{~S}$ and $2 \mathrm{R}$ primers $(10 \mathrm{mmol} / \mathrm{L})$, following the manufacturer's instructions. The PCR conditions consisted of initial denaturation at $94^{\circ} \mathrm{C}$ for 5 minutes, followed by 40 repeated cycles $\left(94^{\circ} \mathrm{C}\right.$ for 30 seconds, $60^{\circ} \mathrm{C}$ for 45 seconds, $72^{\circ} \mathrm{C}$ for 90 seconds) plus $72^{\circ} \mathrm{C}$ for 7 minutes. The HIV-2 V3-loop gene was sequenced using $2 \mathrm{~S}$ and $2 \mathrm{R}$ primers, as in the nested-PCR reactions. Sequencing reactions were performed on an ABI Prism Dye Terminator kit using an automated sequencer (ABI Prism 3130; Applied Biosystems). GenSearch software (Version 19; PhenoSystems) was used to edit and align the nucleotide sequences. When double chromatogram peaks were present, all amino acids possibilities were tested for tropism interpretation. HIV-2 V3-loop sequences were deposited in GenBank with the following accession numbers: JN230718-JN230770.

## V3-Loop Global Net Charge

The gp105 V3-loop global net charge was obtained by the addition of positively charged amino acids (arginine and lysine) minus negatively charged amino acids (aspartic acid and glutamic acid).

## Statistical Analyses

Statistical analyses were performed with StatEL software (Version 2.5; $\mathrm{AD}$ Science). The association of $\mathrm{CD} 4$ cell count with viral tropism was tested by a Mann-Whitney test. The association of proportion of patients with detectable plasma HIV-2 RNA with viral tropism was tested by Fisher's exact test. The association of each mutation in V3 loop with X4 or dual/mixed tropism was analyzed using the $\chi^{2}$ test or Fisher's exact test. Tropism prediction sensitivity was defined as the proportion of phenotypically dual or X4 tropism HIV-2 viruses presenting with mutations of interest among all the isolates analyzed. Tropism prediction specificity was defined as the proportion of phenotypically R5 tropism HIV-2 strains without any mutations of interest among all the isolates analyzed.

## RESULTS

## Tropism Determination by Phenotypic Assay

Results of the phenotypic tropism assay using GHOST(3) cell lines are depicted in Table 1. Cellular viability was $>95 \%$ in all samples tested. All controls of low-level infection showed concordant results, and cytopathogenic effects were observed in all samples displaying positive RTCN values, including the lowest ones (data not shown).

Of the $53 \mathrm{HIV}-2$ clinical isolates tested, 34 were classified as R5 tropic (64\%), 12 as D/M tropic (23\%), and 7 as $\mathrm{X} 4$ tropic (13\%). Among the $7 \mathrm{X} 4$ isolates, 4 exhibited a low-level signal on CCR5 cells. These isolates were considered X4 tropic, because their CCR5 RTCN values were $<15 \%$ of the corresponding CXCR4 RTCN values, as defined in Materials and Methods. Among the $34 \mathrm{R} 5$ isolates, 30 showed a positive signal exclusively on CCR5 cells, and 4 displayed a weak signal on CXCR4 cells. Thus, these 4 isolates were considered R5 tropic, because their CXCR4 RTCN values represented $<5 \%$ of the corresponding CCR5 RTCN value.

## Genotypic Tropism Assay: Analysis of V3-Loop Sequences

Amino acid V3-loop sequences of the 53 HIV-2 clinical isolates are depicted in Table 1. D/M and X4 isolates were pooled. V3-loop sequences issued from R5-tropic isolates $(\mathrm{n}=34)$ were then compared with the V3-loop sequences issued from D/M- or X4- (D/M-X4-) tropic isolates $(\mathrm{n}=19)$ to identify statistically significant differences in residues between the 2 groups of sequences.

Eight positions in the V3-loop sequences were significantly associated with D/M-X4 tropism in this series (Table 2). Any substitution at position 18 was associated with a D/M-X4 tropism in all cases $(P<.00001)$. The remaining mutations associated with $\mathrm{D} / \mathrm{M}-\mathrm{X} 4$ tropism were V19K/R $(P<.00001)$, S22 A/F/Y $(P<.002)$, Q23R $(P<.00001)$, insertions at position $24(P<.00001)$, $\mathrm{I} 25 \mathrm{~L} / \mathrm{Y}(P<.0004)$, R28K $(P<.0004)$, and R30K $(P<.014)$. These mutations showed a specificity of $100 \%$ for D/M-X4 tropism, because they were never found in R5 isolates, except for the $\mathrm{R} 28 \mathrm{~K}$ and $\mathrm{R} 30 \mathrm{~K}$ mutations detected in 4 (specificity, 88\%) and 5 (specificity, 85\%) R5 isolates, respectively. In addition, a V3-loop global net charge of more than +6 was also associated with $\mathrm{D} / \mathrm{M}-\mathrm{X} 4$ tropism $(P<.00001)$. Regarding the V3-loop sequence of the HIV-2 group A reference strain (HIV-2 $2_{\mathrm{ROD}}$ ), this exhibited 2 of the criteria previously associated with D/M-X4 tropism, including a V3 global net charge of +7 and a L18H mutation. None of the genotypic determinants identified in our study seemed to be HIV-2 group dependent.

Table 1. Results of Phenotypic and Genotypic Tropism Tests of 53 HIV-2 Clinical Isolates

| Virus | Group | Phenotypic Tropism in GHOST(3) Cells |  |  | gp105 V3-Loop Sequences |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|  |  | RTCN |  | Viral <br> Tropism | 12345678910 <br> $C K R P G N K T V L$ | 101 <br> $L$ | 111 <br> $P$ |  | 131 <br> $T \quad L$ <br> $T$ | 415 <br> $-M$ | 16 <br> $s$ | 17 <br> $G$ | 18 <br> $L$ |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  | V3-Loop |  |
|  |  | CCR5 | CXCR4 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  | $R$ |  |  |  |  |  | Net <br> Charge | Viral <br> Tropism |
| $10-002$ | A | 36 | $\ldots$ | R5 | CKRPGNKTVA | $A P$ | 1 | T | L | M: |  | G L | L 1 | $\mathrm{~F}$ | $\mathrm{H}$ | $\mathrm{s}$ | . | 0 | $P$ | । | $\mathrm{N}$ | K | R | $P$ | R | $0 A$ | $A W$ | / $\mathrm{C}$ | 6 | R5 |
| $10-003$ | A | 11 | $\ldots$ | R5 | CKRPGNKTVV | $V P$ | 1 | T | L | M | 5 | G L | L V | $F$ | $\mathrm{H}$ | $\mathrm{S}$ | .. ... | 0 | P $\ldots$ | । | N | K | R | P | R | Q $A$ | A W | / $\mathrm{C}$ | 6 | R5 |
| $10-005$ | A | 19 | $\ldots$ | R5 | $C K R P G N K T \vee V$ | $V P$ | 1 | T | L | $M$ : | 5 | G L | L V | $F$ | $\mathrm{H}$ | S | .. ... | 0 | P | । | N | K | R | P | R | Q $A$ | A W | I C | 6 | R5 |
| $10-006$ | A | 10 |  | R5 | CRRPGNKTVV | $V P$ | 1 | T | L | M | s | G L | L V | $\mathrm{VF}$ | $\mathrm{H}$ | S |  | 0 | P $\ldots$ | 1 | $\mathrm{~N}$ | T | R | P | $R C$ | Q $A$ | A W | V C | 5 | R5 |
| $10-007$ | B | 28 | $\ldots$ | R5 | CKRPGNKTVK | $K P$ | 1 | T | 1 | A : | 5 | G L | L L | $-F$ | $\mathrm{H}$ | S | $\cdots$ | 0 | P | । | N | A | к | P | R | Q $A$ | A W | I C | 6 | R5 |
| $10-008$ | B | 9 | $\ldots$ | R5 | $C R R P G N K T \vee V$ | $V P$ | 1 | T | v | M |  | G L | L 1 | $\mathrm{~F}$ | $\mathrm{H}$ | S | $\ldots \ldots$ | 0 | $P \ldots$ | 1 | N | K | R | P | $R$ | Q $A$ | $A W$ | / $\mathrm{C}$ | 6 | R5 |
| $10-009$ | A | 428 | $\ldots$ | R5 | CRRPGNKTVV | $V P$ | 1 | T | L | M: | 5 | G L | L V | V F | $\mathrm{H}$ | S | $\ldots \ldots$ | 0 | P | 1 | N | R | R | P | R | 0 | A W | / $\mathrm{C}$ | 6 | R5 |
| $10-010$ | B | 19 |  | R5 | $C R R P G N K T \vee V$ | $V P$ | v | T | 1 | $M:$ | 5 | G L | L 1 | $\mathrm{~F}$ | $\mathrm{H}$ | S |  | 0 | P $\ldots$ | 1 | $\mathrm{~N}$ | R | R | P | K | Q | A W | l C | 6 | R5 |
| 10-011 | A | 24 | $\ldots$ | R5 | CKRPGNKTVV | $V P$ | 1 | T | L | $M$ : | 5 | G L | L V | V F | $\mathrm{H}$ | $\mathrm{S}$ | ... | 0 | P | 1 | N | R | R | P | K | Q | A W | I C | 6 | R5 |
| 10-015 | A | 66 |  | R5 | CKRPGNKTVV | $V P$ | 1 | T | L | M: | 5 | G L | L V | / $\mathrm{F}$ | $\mathrm{H}$ | $\mathrm{S}$ | $\ldots \ldots$ | 0 | P $\ldots$ | 1 | N | R | R | P | R | o, | A W | / C | 6 | R5 |
| 10-020 | A | 10 | $\ldots$ | R5 | CKRPGNKTVV | $V P$ | 1 | T | L | $M$ | 5 | G L | L V | / F | $\mathrm{H}$ | S | $\ldots \ldots$ | 0 | P $\ldots$ | 1 | $\mathrm{~N}$ | K | R | P | K | Q | A W | I C | 6 | R5 |
| $10-021$ | B | 83 | $\ldots$ | R5 | $C R R P G N K T \vee V$ | $V P$ | 1 | T | v | $M$ | 5 | G L | L 1 | $\mathrm{~F}$ | $\mathrm{H}$ | S | ... | 0 | P | 1 | N | R | R | P | R | $0_{1}$ | $A W$ | / $\mathrm{C}$ | 6 | R5 |
| $10-024$ | $\mathrm{H}$ | 12 | $\ldots$ | R5 | $C R R P G N K T \vee L$ | $L P$ | v | T | 1 | $M$ : | sc | G L | L L | $-F$ | $\mathrm{H}$ | S | ... | 0 | P | 1 | N | N | к | P | K | Q | A W | V $\mathrm{C}$ | 5 | R5 |
| $10-026$ | B | 476 | $\ldots$ | R5 | $C K R P G N K T \vee V$ | $V P$ | 1 | T | v | $M$ : | 5 | G L | L 1 | $\mathrm{~F}$ | $\mathrm{H}$ | S | ... | 0 | P | । | N | T | R | P | R | $Q_{1}$ | A W | I C | 5 | R5 |
| $10-027$ | A | 31 | $\ldots$ | R5 | $C R R P G N K T \vee V$ | $V P$ | 1 | T | L | $M$ : | 5 | G L | L V | V F | $\mathrm{H}$ | S | $\ldots \ldots$ | 0 | P | 1 | N | T | R | P | R | $0^{\prime}$ | A W | I C | 5 | R5 |
| $10-029$ | A | 894 | 8 | R5 | CRRPGNKTVV | $V P$ | 1 | T | L | $M$ : | 5 | G L | L V | V F | $\mathrm{H}$ | S | ... | 0 | P | 1 | N | R | R | P | R | Q | $A W$ | / $\mathrm{C}$ | 6 | R5 |
| $10-031$ | B | 587 | $\ldots$ | R5 | $C R R P G N K T \vee L$ | $L P$ | 1 | T | v | $M$ | 5 | G L | L V | V F | $\mathrm{H}$ | S | ... | 0 | P | 1 | N | T | R | P | R | $0,1+2$ | A W | I C | 5 | R5 |
| $10-033$ | B | 608 | $\ldots$ | R5 | CKRPGNKTVL | $L P$ | 1 | T | v | M | 5 | G L | L L | $-F$ | $\mathrm{H}$ | $\mathrm{S}$ | ... | 0 | P | । | N | T | R | P | R | Q | A W | / $\mathrm{C}$ | 5 | R5 |
| $10-040$ | A | 466 | 14 | R5 | $C K R P G N K T \vee V$ | $V P$ | 1 | T | L | $M$ : | 5 | G L | L V | V F | $\mathrm{H}$ | S | ... | 0 | P | । | N | T | R | P | R | o, | A W | I C | 5 | R5 |
| $10-042$ | A | 110 | $\ldots$ | R5 | CKRPGNKTVL | $L P$ | 1 | T | L | M: |  | G L | L V | / $\mathrm{F}$ | $\mathrm{H}$ | $\mathrm{S}$ |  | 0 | P $\ldots$ | 1 | N | R | R | P | R | o | A W | / $\mathrm{C}$ | 6 | R5 |
| $10-044$ | A | 443 | 7 | R5 | CKRPGNKTVL | $L P$ | v | T | L | $M$ : |  | G L | L V | / $\mathrm{F}$ | $\mathrm{H}$ | S | ... | 0 | P | । | N | K | R | P | R | Q | A W | I C | 6 | R5 |
| $10-046$ | A | 99 |  | R5 | $C K R P G N K T \vee V$ | $V P$ | 1 | T | L | M |  | G L | L V | VF | $\mathrm{H}$ | S |  | 0 | $P \ldots$ | 1 | N | K | R | P | R | o | A W | / $\mathrm{C}$ | 6 | R5 |
| $10-047$ | B | 704 | $\ldots$ | R5 | CKRPGNKTVK | $K P$ | 1 | T | 1 | A : | 5 | G L | L L | - $\mathrm{F}$ | $\mathrm{H}$ | S | $\ldots$ | 0 | P | 1 | N | T | R | P | R | 0 | A W | / $\mathrm{C}$ | 6 | R5 |
| $10-048$ | A | 126 | $\ldots$ | R5 | $C K R P G N K T \vee V$ | $V P$ | 1 | T | L | $M$ : | 5 | G L | L V | $\mathrm{F}$ | $\mathrm{H}$ | S | ... | 0 | P | । | N | R | R | P | R | $0_{1}$ | A W | I C | 6 | R5 |
| $10-050$ | A | 658 | $\ldots$ | R5 | $C K R P G N K T \vee A$ | $A P$ | v | T | L | $M$ : | 5 | G L | L V | $F$ | $\mathrm{H}$ | $S$ | $\ldots \ldots$ | 0 | A | 1 | N | N | R | P | R | $Q_{1}$ | A W | I C | 5 | R5 |
| $10-051$ | A | 373 | $\ldots$ | R5 | CRRPGNKTVV | $V P$ | 1 | T | L | $M$ : | 5 | G L | L 1 | $\mathrm{~F}$ | $\mathrm{H}$ | S | $\ldots \ldots$ | 0 | P | । | N | R | R | P | $\kappa$ | $\mathrm{Q}$ | A W | / $\mathrm{C}$ | 6 | R5 |
| $10-056$ | A | 648 | 32 | R5 | $C R R P G N K T \vee R$ | $R P$ | 1 | T | L | $M$ | 5 | G L | L V | $\mathrm{F}$ | $\mathrm{H}$ | S | $\cdots$ | 0 | P | 1 | N | N | R | P | $R$ | $\mathrm{Q}$ | A W | V $\mathrm{C}$ | 6 | R5 |
| $10-058$ | B | 357 | $\ldots$ | R5 | CKRPGNKTVV | $V P$ | 1 | T | v | M | 5 | G L | L N | $F$ | $\mathrm{H}$ | S | $\ldots$ | 0 | P | । | N | T | K | P | $R$ | $\mathrm{Q}$ | A W | / $\mathrm{C}$ | 5 | R5 |
| $10-060$ | A | 275 | $\ldots$ | R5 | CKRPGNKTVI | I P | v | T | L | $M:$ |  | G L | L V | $\mathrm{F}$ | $\mathrm{H}$ | S | $\cdots$ | 0 | P | 1 | N | K | R | P | R | $\mathrm{Q}$ | A W | I C | 6 | R5 |
| $10-061$ | B | 49 | $\ldots$ | R5 | CKRPGNKTVV | $V P$ | 1 | T | 1 | M: |  | G L | L N | $F$ | H | S | ... | 0 | P | । | N | N | R | P | $R$ | $\mathrm{Q}$ | A W | / $\mathrm{C}$ | 5 | R5 |
| $10-064$ | A | 584 | $\ldots$ | R5 | $C K R P G N K T \vee V$ | $V P$ | v | T | 1 | $M$ : |  | G L | L V | $F$ | H | S | $\ldots$ | 0 | P. | 1 | N | N | K | P | R | $\mathrm{Q}$ | A W | I C | 5 | R5 |
| $10-066$ | A | 53 | $\ldots$ | R5 | $C R R P G N K T \vee E$ | $E P$ | 1 | T | L | F |  | G L | L 1 | $\mathrm{~F}$ | H | S | $\ldots \ldots$ | 0 | A | । | N | Q | R | P | $\kappa$ | $\mathrm{Q}$ | A W | V $\mathrm{C}$ | 5 | R5 |
| $10-067$ | B | 744 | $\ldots$ | R5 | CRRPGNKTVV | $V P$ | 1 | T | 1 | $M$ |  | G L | L N | I $F$ | H | S | $\cdots$ | 0 | P | । | N | T | R | P | R | $\mathrm{Q}$ | A W | I | 5 | R5 |
| $10-074$ | A | 769 |  | R5 | CKRPGNKTV | I P | । | T | L | $M:$ | 5 | G L | L 1 | $\mathrm{~F}$ | $\mathrm{H}$ | S |  | 0 |  | 1 | N | R | R | P | $R$ | $\mathrm{Q}$ | A W | / $\mathrm{C}$ | 6 | R5 |
| 10-028/ROD | A | 97 | 515 | Dual | CKRPGNKTVK | $\mathrm{ko}$ |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  | I C | 7 | Dual-X4 |

Table 1 continued.

<table><tr><td rowspan="3">Virus</td><td rowspan="3">Group</td><th colspan="3">Phenotypic Tropism in GHOST(3) Cells</th><th>gp105 V3-Loop Sequences</th><th></th><th></th></tr><tr><td colspan="2">RTCN</td><td rowspan="2">Viral Tropism</td><td rowspan="2">${\ldots}{\ldots}$ 84 82334 GvHQP1NPQW C</td><td>V3-Loop</td><td></td></tr><tr><td>CCR5</td><td>CXCR4</td><td>Net Charge</td><td>Viral Tropism</td></tr><tr><td>10-004</td><td>A</td><td>5</td><td>25</td><td>Dual</td><td>L MGRRHRPVNP. W C</td><td>8</td><td>Dual-X4</td></tr><tr><td>10-016</td><td>A</td><td>20</td><td>37</td><td>Dual</td><td>LMSGMHCONDINGQP1NKPQW</td><td>6</td><td>Dual-X4</td></tr><tr><td>10-018</td><td>A</td><td>100</td><td>87</td><td>Dual</td><td>TSGHRHRPVN IP1 W</td><td>6</td><td>Dual-X4</td></tr><tr><td>10-023</td><td>A</td><td>98</td><td>232</td><td>Dual</td><td>FBARDINGRP1N( QWC</td><td></td><td>Dual-X4</td></tr><tr><td>10-045</td><td>A</td><td>37</td><td>205</td><td>Dual</td><td>CKRPGNKTVTGHRHQPvTPQWC</td><td>6</td><td>Dual-X4</td></tr><tr><td>10-049</td><td>A</td><td>390</td><td>703</td><td>Dual</td><td>CKRPGNKTVVGRRHRPVNKK QWC</td><td>8</td><td>Dual-X4</td></tr><tr><td>10-052</td><td>B</td><td>45</td><td>232</td><td>Dual</td><td>CRRPGNKTVLTMGQRHBARDINGRPvKKF. WC</td><td>8</td><td>Dual-X4</td></tr><tr><td>10-062</td><td>B</td><td>722</td><td>531</td><td>Dual</td><td>CKRPGNKTVKTMGQRHQP1NKKFQWC</td><td>8</td><td>Dual-X4</td></tr><tr><td>10-065</td><td>A</td><td>133</td><td>398</td><td>Dual</td><td>CRRPGNKTVIP1TMSGQKH${\cdots}{\ldots}$RPVNKFQWC</td><td>8</td><td>Dual-X4</td></tr><tr><td>10-069</td><td>A</td><td>779</td><td>300</td><td>Dual</td><td>CKRPGNKTVVGRHQYNKFQWC</td><td>7</td><td>Dual-X4</td></tr><tr><td>10-070</td><td>B</td><td>290</td><td>494</td><td>Dual</td><td>CRRPGNKTVLP1TLMSGQHa ${\ldots}$RPVLN KKF( QWC</td><td>8</td><td>Dual-X4</td></tr><tr><td>10-012</td><td>A</td><td>9</td><td>103</td><td>X4</td><td>CKRPGNKTVTPTSGQFHRPVKF. WC</td><td></td><td>Dual-X4</td></tr><tr><td>10-013</td><td>A</td><td>7</td><td>56</td><td>X4</td><td>TMGKIH$F,{\ldots}{\ldots}$QPVQC</td><td></td><td>Dual-X4</td></tr><tr><td>10-017</td><td>B</td><td>.</td><td>93</td><td>X4</td><td>CKRPGNKTVVTMSGQKHRP1PQWC</td><td></td><td>Dual-X4</td></tr><tr><td>10-019</td><td>A</td><td> ${\ldots}$ </td><td>9</td><td>X4</td><td>CKRPGNKSVVTLSGYH${\ldots},{\ldots}$QA ${\ldots}$ KPQWC</td><td>7</td><td>Dual-X4</td></tr><tr><td>10-037</td><td>A</td><td>26</td><td>258</td><td>X4</td><td>CKRPGNKTVKPTSGQHQV1NAPQWC</td><td>7</td><td>Dual-X4</td></tr><tr><td>10-054</td><td>A</td><td> ${\ldots}$ </td><td>150</td><td>X4</td><td>CKRPGNKTVEAGHIH$Y{\ldots}{\ldots}$RSGLTPQWC</td><td>7</td><td>Dual-X4</td></tr><tr><td>10-055</td><td>A</td><td>19</td><td>206</td><td>X4</td><td>CRRPGNKTVIPM SGQHBrussRPVLNKPQW C</td><td></td><td>Dual-X4</td></tr></table>

Bold and italicized amino acids are mutations found to be associated in the present study with dual/mixed or X4 tropism.

Abbreviations: HIV-2, human immunodeficiency virus type 2; RTCN, ratio to cell negative.

a HIV-2 reference strains.

Table 2. Sensitivity and Specificity of Mutations in the gp105 V3 Loop Associated With Human Immunodeficiency Virus Type 2 Dual/Mixed or X4 Tropism

| Mutation | $P$ | Sensitivity (95\% <br> CI), \% | Specificity (95\% <br> CI), \% |
| :--- | :--- | :---: | :---: |
| L18Z $^{*}$ | $<.00001$ | $100(82-100)$ | $100(90-100)$ |
| V19 K/R | $<.00001$ | $89(67-99)$ | $100(90-100)$ |
| S22 A/F/Y | $<.001$ | $32(13-57)$ | $100(90-100)$ |
| Q23R | $<.00001$ | $58(34-80)$ | $100(90-100)$ |
| Insertion 24 | $<.00001$ | $74(49-90)$ | $100(90-100)$ |
| I25L/Y | $<.0003$ | $37(16-62)$ | $100(90-100)$ |
| R28K | $<.0003$ | $58(34-80)$ | $88(73-97)$ |
| R30K | $<.01$ | $50(26-74)$ | $82(65-93)$ |
| V3 net charge | $<.00001$ | $79(54-94)$ | $100(90-100)$ |

Abbreviation: $\mathrm{Cl}$, confidence interval.

Thus, if we retained mutations with a sensitivity of $>70 \%$ and a specificity of $100 \%, 4$ major genotypic determinants of D/M-X4 tropism in the HIV-2 V3 loop can be defined: (1) any mutation at residue 18 (sensitivity, 100\%; specificity, 100\%); (2) V19 K/R (sensitivity, 89\%; specificity, 100\%); (3) V3-loop global-net charge (sensitivity, 79\%; specificity, 100\%), and (4) insertions at residue 24 (sensitivity, 74\%; specificity, 100\%). These major genotypic determinants were frequently found associated. Indeed, among the $19 \mathrm{D} / \mathrm{M}-\mathrm{X} 4$ isolates, 4,3 , and 2 major genotypic determinants were present in 10,6, and 3 isolates, respectively. The presence of 2 major genotypic determinants has a positive predictive value of $100 \%$ ( $95 \%$ confidence interval, $82 \%-100 \%)$ and a negative predictive value of $100 \%$ ( $95 \%$ confidence interval, $90 \%-100 \%)$ in this series of HIV-2 clinical isolates.

## Genotypic Tropism Assay: Analysis of V3-Loop Sequences Available in Literature

The reanalysis of distinct data set sequences of HIV-2 V3-loop sequences currently available in the literature and in the Los Alamos National Laboratory (LANL) HIV database is reported in Table 3 [16, 17, 30, 31]. Concerning D/M-X4 HIV-2 isolates $(n=17), \geq 1$ of the major genotypic determinants of CXCR4 coreceptor usage identified in our study was present in all X4 HIV-2 samples $(n=7)$ and in 4 of 10 D/M HIV-2 samples. Concerning the 33 R5 HIV-2 sequences available from other data sets, they showed that none exhibited any of the 4 major genotypic determinants identified in our study. Overall, we observed a good concordance in R5 and X4 tropism predictions. The use of major genotypic determinants identified in our study shows a sensitivity of $65 \%$ and a specificity of $100 \%$ in detecting D/M-X4 viruses in these distinct data sets of HIV-2 sequences.

## Association Between Tropism and Immunovirological Status

Among the 34 patients infected by R5-tropic virus and the 19 patients infected by a D/M- or X4-tropic virus, $\mathrm{CD} 4$ cells counts were available at the time of collection in $25(74 \%)$ and $16(84 \%)$ of the patients, respectively. CD4 cells counts were significantly different between these 2 groups $(P=.002)$, with a median of 286 cells $/ \mathrm{mm}^{3}$ (interquartile range, 218-424) and 66 cells $/ \mathrm{mm}^{3}$ (interquartile range, $29-159$ ) in the $\mathrm{R} 5-$ and D/M-X4-tropic groups, respectively (Table 4). Plasma HIV-2 RNA levels were available in 29 R5-tropic patients (85\%) and in $15 \mathrm{D} / \mathrm{M}$ - or X4-tropic patients (79\%). The proportion of patients with detectable plasma HIV-2 RNA was higher in the D/M-X4-tropic group $(\mathrm{n}=11 ; 73 \%)$ than in the R5-tropic group $(\mathrm{n}=14 ; 48 \%)$ ( Table 4) , but this difference was not statistically significant (P=.10) .

## DISCUSSION

In the present study, which was carried out on $53 \mathrm{HIV}-2$ clinical isolates, we set up a phenotypic tropism assay with an automatic final step based on cytometric analyses to assess infection level. One limitation of this phenotypic tropism test is the absence of parental GHOST cell lines that only express the CD4 receptor, which may provide more reliable determination of the threshold of infection but which are still unavailable at this time from the NIH AIDS Reagent Program. In our study, we identified 4 major V3-loop genotypic determinants of CXCR4 coreceptor use: mutations at residues 18 and 19 , insertions at residue 24 , and the V3-loop global net charge.

Previous studies on the V3-loop genotypic determinants of HIV-2 R5/X4 tropism have been based on smaller numbers of HIV-2 viruses, ranging from 5 to 18 , with very few $\mathrm{X} 4$ and $\mathrm{D} / \mathrm{M}$ viruses [15-17, 29-31]. Moreover, different methods of phenotypic tropism assays were used in these studies, preventing a reliable comparison of the results. Four of these studies showed an association between some V3-loop genotypic determinants and phenotypic tropism $[15,17,29,30]$. No such association was found in the other 2 studies $[16,31]$. In our study, the 4 major genotypic determinants of $\mathrm{D} / \mathrm{M}-\mathrm{X} 4$ tropism were all located in the gp105 V3-loop in its C-terminal part, which is already known to be a critical region for CCR5 or CXCR4 use in HIV-2 [30]. A V3-loop global net charge of more than +6 and the presence of mutations at residues 18 and 19 were previously observed in HIV-2 X4 or D/M viruses. However, the very low number of $\mathrm{D} / \mathrm{M}-\mathrm{X} 4$ viruses in these studies prevents statistical analyses $[15,17,30]$.

Table 3. Results of Phenotypic Tropism, V3-Loop Sequence and Genotypic Tropism Determination of the 51 Human Immunodeficiency Virus (HIV) Type 2 Clinical Isolates Available in the Literature and the LANL HIV Database

gp105 V3-Loop Sequences

<table><tr><td rowspan="2">Virus</td><td rowspan="2">Reference Groups</td><td rowspan="2"></td><td rowspan="2">Phenotypic Determination Models</td><td></td><td>gp105 V3-Loop Sequences</td><td></td><td></td></tr><tr><td>Phenotypic Viral Tropism</td><td>7 18 20 22 . 2324 25 37 2829 31 23 1TLMSGLvFHs${\ldots}{\ldots}$QP1N $T$ RRQwc</td><td>V3-Loop Net Charge</td><td>Genotypic Viral Tropisma</td></tr><tr><td>GH-1</td><td>[30]</td><td>. . . .</td><td>HeLa-CD4 Jurkat, Molt-4</td><td>R5</td><td>LSLFHSAUTIONQP1NTPQW</td><td>5</td><td>R5</td></tr><tr><td>UC1</td><td>[30]</td><td>. . . .</td><td>HeLa-CD4 Jurkat, Molt-4</td><td>R5</td><td>F11MSLNFHS${\ldots}{\ldots}$QPLNTPQWC</td><td>5</td><td>R5</td></tr><tr><td>ST</td><td>[30]</td><td>...</td><td>HeLa-CD4 Jurkat, Molt-4</td><td>R5</td><td>1LSLFHSP1NRPQW</td><td></td><td>R5</td></tr><tr><td>NARI-1</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5CKI</td><td>1LSLFHS.QP1NNPQW</td><td></td><td>R5</td></tr><tr><td>NARI-2</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>P1LSLVFHSQP1NTPQW</td><td></td><td>R5</td></tr><tr><td>NARI-3</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVVP1LSLFHSQP1NNPQW</td><td></td><td>R5</td></tr><tr><td>NARI-4</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVVP1LSLVFHSQP1NTRPQW</td><td></td><td>R5</td></tr><tr><td>NARI-5</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVVP1LSLVFHSQP1NPQ</td><td></td><td>R5</td></tr><tr><td>NARI-6</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>C R R P G N K T V VP1LSLVFHSQP1NDRPRQW</td><td></td><td>R5</td></tr><tr><td>NARI-7</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CRRPGNKTVP1LM SLFHSQP1NKPQW C</td><td></td><td>R5</td></tr><tr><td>NARI-8</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>P1LMSLVFHSQP1NKPQW</td><td></td><td>R5</td></tr><tr><td>NARI-9</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>LM SLVFHSQPNKPQW C</td><td></td><td>R5</td></tr><tr><td>NARI-10</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>HSEXINGLYP</td><td></td><td>R5</td></tr><tr><td>NARI-11</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVVPLSL1FHSQPNTPQW C</td><td></td><td>R5</td></tr><tr><td>NARI-14</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVLS1FHSQPNTPQW</td><td></td><td>R5</td></tr><tr><td>NARI-15</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVVALSLVFHSQP1NAPQW</td><td></td><td>R5</td></tr><tr><td>NARI-16</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVPLSLVFHSQPNKPQW</td><td></td><td>R5</td></tr><tr><td>NARI-17</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVVP1LMSLVFHSQP.. INKPQW C</td><td></td><td>R5</td></tr><tr><td>NARI-18</td><td>[31]</td><td>A</td><td>GHOST</td><td>R5</td><td>CKRPGNKTVVFMLSLFHSQPNNPQW C</td><td></td><td>R5</td></tr><tr><td>A1958</td><td>[16]</td><td>A</td><td>PBMCA32/A32</td><td>R5</td><td>CRRPGNKTVVPVLSGL ${\vee}$ FHSQP.NTPQW</td><td></td><td>R5</td></tr><tr><td>A2267</td><td>[16]</td><td>A</td><td>PBMCA32/A32. $\mathrm{GHOST}(4)$</td><td>R5</td><td>CKBPGNKTVI1LMSLVFHSQPNTPQW</td><td></td><td>R5</td></tr><tr><td>A2270</td><td>[16]</td><td>A</td><td>PBMC $\Delta 32/\Delta 32$ . $\operatorname{GHOST}(4)$</td><td>R5</td><td>CKRPGNKTVV1LSLFHSPNTPQW</td><td></td><td>R5</td></tr><tr><td> $310072$ </td><td> $\lbrack 16\rbrack$ </td><td>B</td><td> $\mathrm{GHOST}(4)$</td><td>R5</td><td>CKRPGNKTVV1VSFHSCORRAQP${\ldots}\quad$NTPQW C</td><td></td><td>R5</td></tr><tr><td>$60415\mathrm{K}$</td><td>[16]</td><td> ${\ldots}$ </td><td>PBMCA32/A32 GHOST(4)</td><td>R5</td><td>CKRPGNKTVT1LSLVFHSP. INTPQW C</td><td></td><td>R5</td></tr><tr><td>SLRHC</td><td>[16]</td><td>A</td><td>$\mathrm{PBMC}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}$ $\mathrm{GHOST}(4)$</td><td>R5</td><td>C B B P G N K T V  K T V  V▪LSLFHSP1NKPQW</td><td></td><td>R5</td></tr><tr><td>2-2</td><td>[17]</td><td>A</td><td>U87. GHOST(3)</td><td>R5</td><td>CKRPGNKTVV1LSLFHSBARDQP${\ldots},$NNPQW C</td><td></td><td>R5</td></tr><tr><td>2-4</td><td> ${}^{\lbrack 17\rbrack}$ </td><td>A</td><td>U87. GHOST(3)</td><td>R5</td><td>CKBPGNKTVVLSLFHSQPNNPQW C</td><td></td><td>R5</td></tr></table>

<table><tr><td rowspan="2">Virus</td><td colspan="2" rowspan="2">Reference Group</td><td rowspan="2">Phenotypic Determination Models</td><td></td><td>gp105 V3-Loop Sequences</td><td></td><td></td></tr><tr><td>Phenotypic Viral Tropism</td><td>12 14 15 16 18 202 22 BARD 23 24 25 262 27 29 31 3233 CKRPGNKTVL1LsGLVHSQP1NPQwC</td><td>V3-Loop Net Charge</td><td>Genotypic Viral Tropism ${}^{\mathrm{a}}$</td></tr><tr><td>1-1</td><td>[17]</td><td>A</td><td>U87, GHOST(3)</td><td>R5</td><td>P1TLMSLVFHS: ...QP1NTRPQW C</td><td></td><td>R5</td></tr><tr><td>1-2</td><td> [17] </td><td>A</td><td>U87, GHOST(3)</td><td>R5</td><td>TLMSFSQP1NTRPQW C</td><td></td><td>R5</td></tr><tr><td>1-3</td><td> [17] </td><td>A</td><td>U87, GHOST(3)</td><td>R5</td><td>1TLMSLFSCONDIAQP1NRPQW C</td><td></td><td>R5</td></tr><tr><td>2-3</td><td> [17] </td><td>A</td><td>U87, GHOST(3)</td><td>R5</td><td>P1TLMSLFHSQP1NTRPQW</td><td></td><td>R5</td></tr><tr><td>2-1</td><td>[17]</td><td>A</td><td>U87, GHOST(3)</td><td>R5</td><td>CKREP1TLSLFHSQP1NKRPQW</td><td></td><td>R5</td></tr><tr><td>4-1</td><td>[17]</td><td>A</td><td>U87, GHOST(3)</td><td>R5</td><td>CKRPP1TLSLFHSQP1NRRPQ W </td><td></td><td>R5</td></tr><tr><td>NARI-12</td><td>[31]</td><td>A</td><td>GHOST</td><td>Dual</td><td>CKREP1LSLFSQP1NPQW C</td><td></td><td>R5</td></tr><tr><td>NARI-13</td><td>[31]</td><td>A</td><td>GHOST</td><td>Dual</td><td>CKREP1TLSLFHSQP1NKRPQW C</td><td></td><td>R5</td></tr><tr><td>310248</td><td>[16]</td><td>A</td><td> PBMCD32/D32,GHOST(4)</td><td>Dual</td><td>C R RP1LSLFSQP1NKPQW</td><td></td><td>R5</td></tr><tr><td>7312A</td><td>[16]</td><td>A/B</td><td> PBMCD32/D32,GHOST(4)</td><td>Dual</td><td>CKRPGNKTVLSFSSTORIANP. INQW</td><td></td><td>R5</td></tr><tr><td>GB87</td><td>[16]</td><td>A</td><td> PBMCD32/D32,GHOST(4)</td><td>Dual</td><td>CRRPGNKTVVP1LSLFHS.P.. INTPQW C</td><td></td><td>R5</td></tr><tr><td>4-3</td><td>[17]</td><td>A</td><td>U87.GHOST(3)</td><td>Dual</td><td>CKRPGNKTVL1TLMSFSQP1NKPQW  C</td><td></td><td>R5</td></tr><tr><td>8-2</td><td> [17] </td><td>A</td><td>U87, GHOST(3)</td><td>Dual</td><td>LSRRS${\ldots}{\ldots}$Q1YQW C</td><td>7</td><td>Dual-X4</td></tr><tr><td>8-3</td><td> [17] </td><td>A</td><td>U87, GHOST(3)</td><td>Dual</td><td>CKRPGNKTVR1TLSRRHSQ1YVNPQW</td><td>7</td><td>Dual-X4</td></tr><tr><td>4-2</td><td> [17] </td><td>A</td><td>U87, GHOST(3)</td><td>Dual</td><td>CKRPGNKTVVP1MLMSFKHS ${\ldots}$ QPVINKPQW</td><td>7</td><td>Dual-X4</td></tr><tr><td>4-4</td><td> [17] </td><td>A</td><td>U87.GHOST(3)</td><td>Dual</td><td>CKRPGNKTVVP1MLMSF${\mathbf{K}}^{{-}} {\vdash}$HSQPVINKPQW</td><td>7</td><td>Dual-X4</td></tr><tr><td>UC2</td><td>[30]</td><td>A</td><td>HeLa-CD4 Jurkat, Molt-4</td><td>X4</td><td>CKRPGNKTVVP1TLSHSAVINKPQW</td><td>7</td><td>Dual-X4</td></tr><tr><td>ROD</td><td>[30]</td><td>A</td><td>Hela-CD4 Jurkat,Molt-4</td><td>X4</td><td>CKRPGNKIVKQ1MMSHFHSHYPCONKPQW</td><td>7</td><td>Dual-X4</td></tr><tr><td>7924A</td><td>[16]</td><td>A</td><td> PBMCD32/D32,GHOST(4)</td><td>X4</td><td>VMSYSCONDIANPVINEPQW</td><td>7</td><td>Dual-X4</td></tr><tr><td>310319</td><td> [16] </td><td>B</td><td> PBMCD32/D32,GHOST(4)</td><td>X4</td><td>CRRPGNKTVV1TSLFSQA1NKPQW</td><td>7</td><td>Dual-X4</td></tr><tr><td>77618</td><td>[16]</td><td>A</td><td> PBMCD32/D32,GHOST(4)</td><td>X4</td><td>CKRPGNKTVLPITLMSGQKFHS . . R P V I NK K P K QAWC</td><td>8</td><td>Dual-X4</td></tr><tr><td>GB122</td><td>[16]</td><td>A</td><td>PBMCD32/D32,GHOST(4)</td><td>X4</td><td>CKRPGNKTVVMLMSQYHFRPVINDPQW C</td><td></td><td>Dual-X4</td></tr><tr><td>8-4</td><td>[17] </td><td>A</td><td>U87. GHOST(3)</td><td>X4</td><td>CKRPGNKTVR1TLSRRHSQ1YVNPQW C</td><td>7</td><td>Dual-X4</td></tr><tr><td>8-1</td><td> [17] </td><td>A</td><td>U87, GHOST(3)</td><td>X4</td><td>LSRRFHSCARIANQ1CONDYVPQW C</td><td>7</td><td>Dual-X4</td></tr></table>

Abbreviation: LANL, Los Alamos National Laboratory.

a Genotypic viral tropism interpretation based on the presence of $\geq 1$ of the CXCR4-use major genotypic determinants described in the present study.

Table 4. CD4 Cell Counts Distribution and Proportion of Patients With Detectable Plasma HIV-2 RNA (Viremic Patients) According to Tropism

| Tropism Group | Median CD4 <br> Cell Count, <br> Median (IQR), cells $/ \mathrm{mm}^{3}$ | Viremic Patients, <br> $\%^{\mathrm{a}}$ |
| :---: | :---: | :---: |
| R5 tropic | $286(218-424)(n=25)$ | $48(n=29)$ |
| D/M tropic | $66(41-152)(n=8)$ | $66(n=9)$ |
| X4 tropic | $66(30-149)(n=8)$ | $80(n=6)$ |
| D/M-X4 tropic ${ }^{b}$ | $66(29-159)(n=16)$ | $73(n=15)$ |

Abbreviations: D/M, dual/mixed; HIV-2, human immunodeficiency virus type 2; $I Q R$, interquartile range.

a Viremic patients were defined as those with plasma HIV-2 RNA levels $>100$ copies/mL.

${ }^{b}$ D/M-X4-tropic group represents pooled data for the D/M- and X4-tropic groups.

Regarding HIV-1 infection, the gp120 V3-loop major genotypic determinants of CXCR4 coreceptor use are at positions 11 and 25 and the V3-loop global net charge; the latter is also described in HIV-2 infection. Conversely, positions 11 and 25 were not found associated with HIV-2 tropism prediction. However, the large genotypic differences between HIV-1 and HIV-2 V3 loops render it difficult to make such comparisons.

Regarding viral tropism analysis of HIV-2 gp105 V3-loop sequences with a known coreceptor usage available in the literature $[16,17,30,31]$, a sensitivity of $65 \%$ and a specificity of $100 \%$ in detecting $\mathrm{D} / \mathrm{M}-\mathrm{X} 4$ viruses was shown using the major genotypic determinants of CXCR4 usage we identified in our study. In all HIV-2 clinical isolates from our series, a mutation at residue 18 is sufficient to predict D/M-X4 tropism. However, among the 7 X4 isolates from the previous study data set, 6 displayed a mutant residue at residue 18; the remaining 1 exhibited only a global net charge of +7 [16]. This suggests that residue 18 alone is probably not sufficient to predict CXCR4 coreceptor usage. In this reanalysis, the lower concordance was observed in samples exhibiting a D/M-tropic mixture, because 6 of the 10 reported $\mathrm{D} / \mathrm{M}$ - tropic viruses were classified as $\mathrm{R} 5$ by analysis with the 4 major genotypic determinants. This observation may be due to the various phenotypic tropism assays used in these studies and to the possible differences in double chromatogram peaks interpretation. Similar rates of sensitivity and specificity were obtained for tropism genotypic prediction in HIV-1 infection. Thus, in the ANRS GenoTropism Study, which evaluated the genotypic prediction of HIV-1 coreceptor use versus the phenotypic Trofile assay, the highest sensitivity of genotypic-tropism prediction was $\sim 60 \%$ according to different genotypic algorithms [38]. For all algorithms tested, the specificity to detect X4 compared with the Trofile test ranged from $79.3 \%$ to $98.5 \%$ [38]. Similar results of genotypic prediction compared to the standard Trofile assay have been reported after analysis of the MOTIVATE-1 study database [39].

In conclusion, in our study we established, for the first time, a strong association between HIV-2 phenotypic tropism and V3-loop genotypic determinants based on a large series of patients offering the opportunity to identify R5- and/or X4-tropic viruses. This genotypic tropism test could help improve the knowledge on HIV-2 pathogenesis by describing the evolution of tropism during the natural history of infection and also by describing the tropism in different compartments.

## Notes

Acknowledgments. C. C., F. D., S. M., F. B. V., and D. D. contributed to the study concept. B. V., A. S., and G. C. performed genotypic tests. B. V., M. H. N., A. S., and G. C. performed phenotypic tests. C. C., L. L., F. D., S. M., and D. D. contributed to the analysis and interpretation of the data. C. C., B. V., F. D., F. B. V., and D. D. contributed to the writing of the manuscript. All authors contributed to the critical reviewing of the manuscript.

Financial support. This work was supported by the French National Agency for Research on AIDS and Viral Hepatitis (ANRS) and by the European Community's Seventh Framework Programme (grant FP7/ 2007-2013) under the project Collaborative HIV and Anti-HIV Drug Resistance Network (CHAIN) (grant 223131).

Potential conflicts of interest. All authors: No reported conflicts.

All authors have submitted the ICMJE Form for Disclosure of Potential Conflicts of Interest. Conflicts that the editors consider relevant to the content of the manuscript have been disclosed.

## References

1. Valadas E, França L, Sousa S, Antunes F. 20 years of HIV-2 infection in Portugal: trends and changes in epidemiology. Clin Infect Dis 2009; 48:1166-7.
2. Brunet S, Thierry D, Barin F, et al. Surveillance de l'infection à VIH-sida en France, 2007. BEH 2008; 45:434-42.
3. Roquebert B, Damond F, Collin G, et al. HIV-2 integrase gene polymorphism and phenotypic susceptibility of HIV-2 clinical isolates to the integrase inhibitors raltegravir and elvitegravir in vitro. J Antimicrob Chemother 2008; 62:914-20.
4. Ntemgwa ML, d'Aquin Toni T, Brenner BG, Camacho RJ, Wainberg MA. Antiretroviral drug resistance in human immunodeficiency virus type 2. Antimicrob Agents Chemother 2009; 53:3611-9.
5. Ren J, Bird LE, Chamberlain PP, Stewart-Jones GB, Stuart DI, Stammers DK. Structure of HIV-2 reverse transcriptase at 2.35-A resolution and the mechanism of resistance to non-nucleoside inhibitors. Proc Natl Acad Sci U.S.A 2002; 99:14410-5.
6. Witvrouw M, Pannecouque C, Switzer WM, Folks TM, De Clercq E, Heneine W. Susceptibility of HIV-2, SIV and SHIV to various anti-HIV-1 compounds: implications for treatment and postexposure prophylaxis. Antivir Ther 2004; 9:57-65.
7. Poveda E, Briz V, Soriano V. Enfuvirtide, the first fusion inhibitor to treat HIV infection. AIDS Rev 2005; 7:139-47.
8. Desbois D, Roquebert B, Peytavin G, et al. In vitro phenotypic susceptibility of human immunodeficiency virus type 2 clinical isolates to protease inhibitors. Antimicrob Agents Chemother 2008; 52:1545-8.
9. Willey S, Peters PJ, Sullivan WM, Dorr P, Perros M, Clapham PR. Inhibition of CCR5-mediated infection by diverse R5 and R5X4 HIV and SIV isolates using novel small molecule inhibitors of CCR5: effects of viral diversity, target cell and receptor density. Antiviral Res 2005; 68:96-108,
10. Armstrong-James D, Stebbing J, Scourfield A, et al. Clinical outcome in resistant $\mathrm{HIV}-2$ infection treated with raltegravir and maraviroc. Antiviral Res 2010; 86:224-6.
11. Stegmann S, Manea ME, Charpentier C, et al. Foscarnet as salvage therapy in HIV-2-infected patient with antiretroviral treatment failure. J Clin Virol 2010; 47:79-81.
12. McKnight A, Dittmar MT, Moniz-Periera J, et al. A broad range of chemokine receptors are used by primary isolates of human immunodeficiency virus type 2 as coreceptors with CD4. J Virol 1998; 72:4065-71.
13. Mörner A, Björndal A, Leandersson A, Albert J, Björling E, Jansson M. CCR5 or CXCR4 is required for efficient infection of peripheral blood mononuclear cells by promiscuous human immunodeficiency virus type 2 primary isolates. AIDS Res Hum Retroviruses 2002; 18:193-200.
14. Guillon C, van der Ende ME, Boers PH, Gruters RA, Schutten M, Osterhaus AD. Coreceptor usage of human immunodeficiency virus type 2 primary isolates and biological clones is broad and does not correlate with their syncytium-inducing capacities. J Virol 1998; 72:6260-3.
15. Mörner A, Björndal A, Albert J, et al. Primary human immunodeficiency virus type 2 (HIV-2) isolates, like HIV-1 isolates, frequently use CCR5 but show promiscuity in coreceptor usage. J Virol 1999; 73:2343-9.
16. Owen SM, Ellenberger D, Rayfield M, et al. Genetically divergent strains of human immunodeficiency virus type 2 use multiple coreceptors for viral entry. J Virol 1998; 72:5425-32.
17. Shi Y, Brandin E, Vincic E, et al. Evolution of human immunodeficiency virus type 2 coreceptor usage, autologous neutralization, envelope sequence and glycosylation. J Gen Virol 2005; 86:3385-96.
18. Blaak H, Boers PH, Gruters RA, Schuitemaker H, van der Ende ME, Osterhaus AD. CCR5, GPR15, and CXCR6 are major coreceptors of human immunodeficiency virus type 2 variants isolated from individuals with and without plasma viremia. J Virol 2005; 79:1686-700.
19. Zhang Y, Lou B, Lal RB, Gettie A, Marx PA, Moore JP. Use of inhibitors to evaluate coreceptor usage by simian and simian/human immunodeficiency viruses and human immunodeficiency virus type 2 in primary cells. J Virol 2000; 74:6893-10.
20. Kuhmann SE, Hartley O. Targeting chemokine receptors in HIV: a status report. Annu Rev Pharmacol Toxicol 2008; 48:425-61.
21. Tsibris AM, Kuritzkes DR. Chemokine antagonists as therapeutics: focus on HIV-1. Annu Rev Med 2007; 58:445-59.
22. Whitcomb JM, Huang W, Fransen S, et al. Development and characterization of a novel single-cycle recombinant-virus assay to determine human immunodeficiency virus type 1 coreceptor tropism. Antimicrob Agents Chemother 2007; 51:566-75.
23. Jensen MA, van 't Wout AB. Predicting HIV-1 coreceptor usage with sequence analysis. AIDS Rev 2003; 5:104-12.
24. Beerenwinkel N, Sing T, Lengauer T, et al. Computational methods for the design of effective therapies against drug resistant HIV strains. Bioinformatics 2005; 21:3943-50.
25. Web PSSM. http://indra.mullins.microbiol.washington.edu/webpssm/ index.html. Accessed 10 March 2011.
26. Geno2pheno coreceptor. http://coreceptor.bioinf.mpi-inf.mpg.de/ index.php. Accessed 10 March 2011.
27. Skrabal K, Low AJ, Dong W, et al. Determining human immunodeficiency virus coreceptor use in a clinical setting: degree of correlation between two phenotypic assays and a bioinformatic model. J Clin Microbiol 2007; 45:279-84.
28. Raymond S, Delobel P, Mavigner M, et al. Correlation between genotypic predictions based on V3 sequences and phenotypic determination of HIV-1 tropism. AIDS 2008; 22:F11-6.
29. Albert J, Stålhandske P, Marquina S, et al. Biological phenotype of HIV type 2 isolates correlates with V3 genotype. AIDS Res Hum Retroviruses 1996; 12:821-8.
30. Isaka Y, Sato A, Miki S, et al. Small amino acid changes in the V3 loop of human immunodeficiency virus type 2 determines the coreceptor usage for CXCR4 and CCR5. Virology 1999; 264:237-43.
31. Kulkarni S, Tripathy S, Agnihotri K, et al. Indian primary HIV-2 isolates and relationship between V3 genotype, biological phenotype and coreceptor usage. Virology 2005; 337:68-75.
32. Davis HE, Rosinski M, Morgan JR, Yarmush ML. Charged polymers modulate retrovirus transduction via membrane charge neutralization and virus aggregation. Biophys J 2004; 86:1234-42.
33. Ho WZ, Cherukuri R, Ge SD, et al. Centrifugal enhancement of human immunodeficiency virus type 1 infection and human cytomegalovirus gene expression in human primary monocyte/macrophages in vitro. J Leukoc Biol 1993; 53:208-12.
34. Pietroboni GR, Harnett GB, Bucens MR. Centrifugal enhancement of human immunodeficiency virus (HIV) and human herpesvirus type 6 (HHV-6) infection in vitro. J Virol Methods 1989; 24:85-90.
35. Vödrös D, Tscherning-Casper C, Navea L, Schols D, De Clercq E, Fenyö EM. Quantitative evaluation of HIV-1 coreceptor use in the GHOST3 cell assay. Virology 2001; 291:1-11.
36. Cho MW, Lee MK, Carney MC, Berson JF, Doms RW, Martin MA. Identification of determinants on a dualtropic human immunodeficiency virus type 1 envelope glycoprotein that confer usage of CXCR4. J Virol 1998; 72:2509-15.
37. Schols D, Esté JA, Cabrera C, De Clercq E. T-cell-line-tropic human immunodeficiency virus type 1 that is made resistant to stromal cellderived factor lalpha contains mutations in the envelope gp120 but does not show a switch in coreceptor use. J Virol 1998; 72:4032-7.
38. Recordon-Pinson P, Soulié C, Flandre P, et al. Evaluation of the genotypic prediction of HIV-1 coreceptor use versus a phenotypic assay and correlation with the virological response to maraviroc: the ANRS GenoTropism study. Antimicrob Agents Chemother 2010; 54:3335-40.
39. McGovern RA, Thielen A, Mo T, et al. Population-based V3 genotypic tropism assay: a retrospective analysis using screening samples from the A4001029 and MOTIVATE studies. AIDS 2010; 24:2517-25.

