# Identification of methylated proteins by protein arginine $\mathrm{N}$-methyltransferase 1, PRMT1, with a new expression cloning strategy 

Kazuhiro Wada, Koichi Inoue, Masatoshi Hagiwara*<br>Department of Functional Genomics, Medical Research Institute, Tokyo Medical and Dental University, 1-5-45 Yushima, Bunkyo, Tokyo 113-8510, Japan

Received 13 December 2001; received in revised form 11 March 2002; accepted 4 April 2002

## Abstract

Methylation at arginines has recently come to attention as a posttranslational modification of proteins, which is implicated in processes from signaling, transcriptional activation, to mRNA processing. Here we report that several proteins extracted from HeLa cells were methylated by PRMT1 (protein arginine $N$-methyltransferease 1) even on a nitrocellulose membrane, while proteins from Escherichia coli are not methylated with this protein. Screening PRMT1 substrates from a $\lambda$ gt11-HeLa cDNA library, we found that more than half of the 48 cDNA clones obtained encode putative RNA-binding proteins that have RGG (arginine-glycine-glycine) motifs, such as hnRNP R (heterogeneous nuclear ribonucleoprotein R) and hnRNP K. We cloned two novel arginine methylation substrates, ZF5 (zinc finger 5) and p137GPI (GPI-anchor protein p137), which do not possess typical RGG motifs. We also cloned a novel protein that has RGG motifs, but does not have any other RNA-binding motifs. We tentatively termed this clone SAMT1 (substrate of arginine methyl transferase 1). $\mathrm{A}_{63}-\mathrm{VLD}_{-65}$ to AAA mutation of PRMT1 suppressed the methylation of recombinant SAMT1 and other RGG proteins in the HeLa extracts. This systematic screening of substrate proteins with the solid phase methylation reaction will contribute to identify new roles of PRMT family. (C) 2002 Elsevier Science B.V. All rights reserved.

Keywords: Arginine methylation; PRMT1; RGG motif; RNA-binding protein; Solid phase screening

## 1. Introduction

Methylation of arginine residues by covalent modifications occurs in eukaryotic proteins during or shortly following mRNA to protein translation. Evidence for posttranslational methylation of arginine residues was first provided by the presence of radioactive species chromatographing at positions near that of arginine in acid hydrolysates of isolated calf thymus nuclei proteins incubated with [methyl $\left.{ }^{14} \mathrm{C}\right]$-S-adenosyl-L-methionine (SAM) [1]. The products of the methylation reaction were determined to be $N^{\mathrm{G}}$-monomethylarginine and $N^{\mathrm{G}}, N^{\mathrm{G}}$-dimethylarginine [2]. Protein arginine $N$-methyltransferases (PRMTs) catalyze the sequential transfer of methyl groups from SAM to the guanidino nitrogens of arginine residues within proteins $[3,4]$. It now appears that there are at least two distinct classes of protein arginine methyltransferases according to their substrate specificity and reaction products [4]. The type I enzymes catalyze the formation of $N^{\mathrm{G}}$-monomethy-larginine and asymmetric $N^{\mathrm{G}}, N^{\mathrm{G}}$-dimethylarginine residues, while type II enzymes catalyze the formation of $N^{\mathrm{G}}$-monomethylarginine and symmetric $N^{\mathrm{G}}, N^{\prime \mathrm{G}}$-dimethylarginine. The asymmetric dimethylarginine appears to be the more common form while symmetric dimethylarginine has so far been identified only in a few proteins, including myelin basic protein (MBP) [5], and spliceosomal $\mathrm{Sm}$ proteins D1 and D3 (SmD1 and SmD3) [6]. Reported type I enzymes are the yeast $\mathrm{Hmtlp} / \mathrm{Rmtlp}$ [7] and the mammalian PRMT1 [8], PRMT3 [9], and CARM1/PRMT4 [10]. PRMT1 and yeast $\mathrm{Hmtlp} / \mathrm{Rmtlp}$ prefer to methylate arginine residues in glycine rich regions, which are found in several RNPs, fibrillarin, and nucleolin [11]. However, CARM1/PRMT4 has little or no activity with substrates preferred by these enzymes, and the best substrate reported for CARM1/ PRMT4 is histone H3 [10]. JBP1/PRMT5 can methylate histone $\mathrm{H} 2 \mathrm{~A}$ and $\mathrm{H} 4$ and myelin basic protein (MBP) [11]. Recently, a potential candidate for a type II enzyme, JBP1/ PRMT5, has been discovered in mammalian cells with an apparent homologue in yeast, Hsl7 [12]. Yet, its sequence is well conserved with other family members, the catalytic activity of PRMT2 has not yet been demonstrated [13]. The conservation among protein arginine $\mathrm{N}$-methyltransferases may underscore the potential biological importance of this posttranslational modification. In addition to these, the type III enzyme was also recently discovered in yeast, and this catalyzes the monomethylation of the internal $\delta$-guanidino nitrogen atom of arginine residues [14].

PRMT1 knockout mouse has already been established, and the phenotype of the homozygous null mutant is embryonic lethal [15], suggesting that PRMT1 is required for very early stages of mouse development. Though recent evidence has implicated arginine-specific protein methyltransferases in nuclear export of some hnRNP proteins and in several signaling pathways (reviewed in Ref. [4]), specific roles of protein methylation and the relevant protein substrates have not been identified. Because each methyltransferase recognizes a different set of protein substrates, systematic analysis of their substrate specificity is now required for differentiating their function. To this end, we report here a new expression cloning strategy to identify substrate proteins of PRMT1, which is potentially applicable to isolate substrates specific for each member of the PRMT family.

## 2. Materials and methods

### 2.1. In vitro methylation reaction

In vitro methylation assay was performed as described by Najbauer et al. [16] with a few modifications. One microgram each of recombinant proteins was added in a $20 \mu \mathrm{l}$ reaction mix containing $50 \mathrm{mM} \mathrm{Na}$-morpholinepropanesul- fonic acid [MOPS] (pH 7.2), $300 \mathrm{mM}, \mathrm{NaCl}, 2 \mathrm{mM}$ ethylenediaminetetraacetic acid [EDTA], $1 \mathrm{mM}$ phenylmethylsulfonyl fluoride [PMSF], $0.1 \mu \mathrm{g} / \mathrm{ml}$ leupeptin and $0.4 \mu \mathrm{M} S$ adenosyl-L- $\left[\right.$ methyl- $\left.{ }^{3} \mathrm{H}\right]$ methionine $\left[{ }^{3} \mathrm{H}-\mathrm{SAM}\right]$ (specific activity of $1.3 \mathrm{Ci} / \mathrm{mmol}$; Amersham Biosciences, NJ) with 0.02-0.04 $\mu \mathrm{g} / \mu 1$ recombinant PRMT1 or bovine serum albumin [BSA] (as a negative control) and incubated at 30 ${ }^{\circ} \mathrm{C}$ for $45 \mathrm{~min}$. The methylation reaction was terminated by adding $20 \mu 1$ sodium dodecyl sulfate [SDS] sample buffer and boiled for $5 \mathrm{~min}$. Each $20 \mu 1$ reaction solution was then loaded onto a $15 \%$ SDS-polyacrylamide gel, and separated by electrophoresis. Gels were stained with Coomassie blue R250 to visualize protein bands and then fluorographed, dried and exposed to Kodak film at $-80^{\circ} \mathrm{C}$ for 2 days.

### 2.2. Solid phase methylation assay

Cell extracts were size-separated on SDS-polyacrylamide gel electrophoresis [SDS-PAGE] and transferred to a nitrocellulose membrane. The membrane was then incubated in a reaction mix containing $50 \mathrm{mM}$ Na-MOPS (pH 7.2), 300 $\mathrm{mM} \mathrm{NaCl}, 2 \mathrm{mM}$ EDTA, $1 \mathrm{mM}$ PMSF, $0.1 \mu \mathrm{g} / \mathrm{ml}$ leupeptin, and $0.4 \mu \mathrm{M}{ }^{3} \mathrm{H}$-SAM with $1 \mu \mathrm{g}$ of recombinant PRMT1 at $30{ }^{\circ} \mathrm{C}$ for $1 \mathrm{~h}$, washed with PBS [phosphate-buffered saline] and TPBS [Tris-phosphate-buffered saline], fluorographed, dried, and exposed to Kodak film at $-80^{\circ} \mathrm{C}$ for 2 days.

### 2.3. Screening of a cDNA library by solid-phase methylation

The HeLa UNI-ZAP cDNA expression library (Stratagene, CA) was plated on Escherichia coli XL-1 blue strain at a density of $1.8 \times 10^{4}$ plaques per $100 \times 140 \mathrm{~mm}$ square agar plate. After incubation for $3 \mathrm{~h}$ at $42^{\circ} \mathrm{C}$, the plates were overlaid with nitrocellulose membrane filters (PROTRAN, Schleicher \& Schuell, NH) that had been soaked with $10 \mathrm{mM}$ isopropyl- $\beta$-D-thiogalactopyranoside [IPTG]. The plates were incubated at $37{ }^{\circ} \mathrm{C}$ for another $4.5-5 \mathrm{~h}$, and their filters were peeled off, immersed in blocking solution [5\% skim milk in PBS] and gently agitated at $4{ }^{\circ} \mathrm{C}$ for $1 \mathrm{~h}$.

![](https://cdn.mathpix.com/cropped/2024_04_28_1c2a0b05acc9b6814cc7g-02.jpg?height=670&width=1584&top_left_y=1614&top_left_x=250)

Fig. 1. Detection of methylated proteins by PRMT1 on a nitrocellulose membrane. (A) Nuclei extracts and cytosolic fractions prepared from HeLa cells cultivated for 3 days in either the absence (lanes 1 and 2) or presence (lanes 3, 4, 5, and 6) of the methyltransferase inhibitor adenosine dialdehyde (AdOX), blotted on a nitrocellulose membrane, and incubated with (lanes 1, 2, 3, and 4) or without (lanes 5 and 6) PRMT1. (B) Whole E. coli proteins from IPTGinduced GST-hnRNP A1 were separated on SDS-PAGE and transferred on a nitrocellulose membrane (lanes 2, 4 and 6). IPTG-untreated $E$. coli proteins were similarly prepared as negative controls (lanes 1,3, and 5). The nitrocellulose membranes were incubated in the absence (lane 1 and 2) or presence (lanes 3 and 4) of recombinant PRMT1 in methylation reaction buffer including ${ }^{3} \mathrm{H}$-SAM at $30^{\circ} \mathrm{C}$ for $1 \mathrm{~h}$. The protein amount of each lane was normalized as shown in the Coomassie staining (lanes 5 and 6).

![](images\imageA3.png)

Fig. 2. Expression cloning protocol of substrate of PRMT1. (A) Brief protocol of the screening of PRMT1 substrates from a $\lambda$ gt $11-\mathrm{HeLa}$ cDNA library. (B) Representative result of the first screening (upper panel). Positive clones indicated by arrows (I, II, and III of the upper panel) were further condensed to monoclonal plaques through a second (not shown) and third screening (lower panels).

Table 1

Summary of expression screening for PRMT1 substrates

<table>
  <thead>
    <tr>
      <th>Gene</th>
      <th>GenBank accession</th>
      <th>Cloned number</th>
      <th>Identified function</th>
      <th>Number of a.a.</th>
      <th>RGG domain (stripes)</th>
    </tr>
    <tr>
      <th>hnRNPR</th>
      <th>AF000364</th>
      <th>9</th>
      <th>RNA binding protein</th>
      <th>633</th>
      <td rowspan="10">![](images\imageA12.png)</td>
    </tr>
    <tr>
      <th>GRY-RBP</th>
      <th>AF037448</th>
      <th>1</th>
      <th>RNA binding protein</th>
      <th>623</th>
    </tr>
    <tr>
      <th>hnRNP K</th>
      <th>S74678</th>
      <th>1</th>
      <th>RNA binding protein</th>
      <th>464</th>
    </tr>
    <tr>
      <th>TAFII 68</th>
      <th>X98893</th>
      <th>3</th>
      <th>TBP ASSOCIATED factor</th>
      <th>589</th>
    </tr>
    <tr>
      <th>RBP58</th>
      <th>U51334</th>
      <th>1</th>
      <th>putative RNA binding protein</th>
      <th>592</th>
    </tr>
    <tr>
      <th>TLS/FUS</th>
      <th>S62140/ X71428</th>
      <th>2</th>
      <th>RNA binding protein</th>
      <th>526</th>
    </tr>
    <tr>
      <th>SAMTI</th>
      <th>AF151813</th>
      <th>8</th>
      <th>unknown</th>
      <th>409</th>
    </tr>
    <tr>
      <th>ZF 5</th>
      <th>D89859</th>
      <th>1</th>
      <th>transcription factor</th>
      <th>449</th>
    </tr>
    <tr>
      <th>p137GPI</th>
      <th>NM 005898</th>
      <th>4</th>
      <th>GPI-anchor protein//membrane component</th>
      <th>649</th>
  </tr>
</table>

The filters were then washed three times for $20 \mathrm{~min}$ in PBS, incubated for $30 \mathrm{~min}$ at $4{ }^{\circ} \mathrm{C}$ in the methylation buffer (50 mM Na-MOPS [pH 7.2], 2 mM EDTA, $0.3 \mathrm{M} \mathrm{NaCl}, 0.1 \mu \mathrm{g} /$ $\mathrm{ml}$ leupeptin, $1 \mathrm{mM}$ PMSF, $0.4 \mu \mathrm{M}{ }^{3} \mathrm{H}-\mathrm{SAM}$; specific activity of $1.3 \mathrm{Ci} / \mathrm{mmol}$ ) with $0.02-0.04 \mu \mathrm{g} / \mu$ l of recombi- nant PRMT1 and further incubated at $30^{\circ} \mathrm{C}$ for $2-3 \mathrm{~h}$ for the solid-phase methylation with gentle agitation. The filters were then washed with TPBS three times for $20 \mathrm{~min}$ and air dried. The filters were fluorographed with $\mathrm{EN}^{3} \mathrm{HANCE}$ spray (Dupont, DE), dried and exposed to $\mathrm{X}$ ray film (BioMax MR-2, Kodak, NY) at $-80^{\circ} \mathrm{C}$ for 1 week. At the first screening, 607 positive clones were detected in $2.0 \times 10^{5}$ plaques. We picked up 100 of these clones and succeeded to isolate 83 positive clones by the third screening. Positive clones were converted to plasmids with the Exassist/Solr E. coli system (Stratagene).

(A)

![](https://cdn.mathpix.com/cropped/2024_04_28_1c2a0b05acc9b6814cc7g-04.jpg?height=1798&width=1201&top_left_y=622&top_left_x=444)

Fig. 3. Primary structure and expression of SAMT1. (A) Predicted amino acid sequence of SAMT1. RGG motifs are indicated by black lines. (B) Expression of SAMT1 in various human tissues. Human multiple tissue Northern blot I or II (Clontech) was hybridized with a probe of full-length SAMT1 cDNA. The tissues from which the RNA is derived are indicated above and size standards (in kb) are indicated to the left.

![](https://cdn.mathpix.com/cropped/2024_04_28_1c2a0b05acc9b6814cc7g-05.jpg?height=719&width=1171&top_left_y=217&top_left_x=427)

Fig. 3 (continued).



### 2.4. Northern hybridization analysis

SAMT full-length cDNA probe or $\beta$-actin probe was labeled with radioactive $\left[\alpha^{-32} \mathrm{P}\right] \mathrm{dCTP}$ using the multiprime DNA labeling system (Amersham Biosciences). A human multiple tissue Northern blot (Clontech, CA) membranes were used. Hybridization was carried out at $42{ }^{\circ} \mathrm{C}$ overnight in hybridization mixture $(6 \times \operatorname{SSC}[1 \times$ $\mathrm{SSC}$ is $0.15 \mathrm{M} \mathrm{NaCl}$ plus $0.015 \mathrm{M}$ sodium citrate], $50 \%$ formamide, $1 \%$ SDS, $1 \times$ Denhardt's solution, $10 \%$ dextran sulfate, $100 \mu \mathrm{g}$ of denatured salmon sperm DNA per $\mathrm{ml}$ ). The membrane was washed three times at room temperature in washing buffer $(0.1 \times \mathrm{SSC}, 0.1 \% \mathrm{SDS})$, and the hybridized transcripts were observed with a BAS2000 image analyzer (Fuji Film, Tokyo).

### 2.5. Cell culture and transfection assays

HeLa cells were grown on cover slips in Dulbecco's modified Eagle's medium [DMEM] supplemented with $10 \%$ calf serum, $100 \mathrm{U} / \mathrm{ml}$ penicillin, and $100 \mathrm{U} / \mathrm{ml}$ streptomycin. Expression vectors of pGFP (Clontech) containing cloned full-length cDNAs were introduced into cells at approximately $60 \%$ confluence with LipofectAMINE (Life Technologies, NY) following the transfection conditions as recommended by the manufacturer. For immunoflourescence, cells were washed in PBS and fixed for $15 \mathrm{~min}$ with $4 \%$ paraformaldehyde in PBS at room temperature, permeabilized, and blocked in PBS containing $0.4 \%$ Triton X-100, $1.5 \% \mathrm{BSA}$, and $5 \%$ normal goat serum.

## 3. Results

### 3.1. Detection of methylated proteins by PRMTI on a nitrocellulose membrane

First we examined whether proteins in HeLa cell extracts could be methylated by PRMT1 on a nitrocellulose membrane. HeLa cells were cultivated for 3 days in the presence of the methyltransferase inhibitor, 20 $\mu \mathrm{M}$ adenosine dialdehyde (AdOX), to prevent endogenous methylation that would mask our assay. Nuclei and cytosolic fractions were then size-separated on SDSPAGE, transferred to a nitrocellulose membrane, and incubated with recombinant human PRMT1 and the methylation substrate ${ }^{3} \mathrm{H}-\mathrm{SAM}$ at $30{ }^{\circ} \mathrm{C}$ for $30 \mathrm{~min}$. Four major bands and seven minor bands in the nuclei fraction appeared after 2 days exposure (Fig. 1A, lane 3), whereas four major bands and three minor bands appeared in the cytosolic fraction (Fig. 1A, lane 4). All these bands were faint or absent in extracts prepared from HeLa cells cultivated without AdOX (Fig. 1A, lanes 1 and 2), suggesting that most arginines in these proteins were already methylated in vivo in HeLa cells. To further examine the validity of the in vitro methylation reaction on the nitrocellulose membrane, we methylated recombinant hnRNP A1 under the control of an IPTG promoter in E. coli. Proteins were isolated from IPTG-treated or -untreated whole $E$. coli containing recombinant hnRNAP Al, separated on SDS-PAGE and transferred to a nitrocellulose membrane. Methylated proteins were specifically detected in the IPTG-induced $E$. coli containing hnRNP A1 (Fig. 1B, lanes 3 and 4). In E. coli not containing human hnRNP A1, no positive band was found in IPTGtreated E. coli (Fig. 1B, lanes 1 and 2). These results show in vitro methylation by PRMT1 is specific to the human protein and that $E$. coli does not methylate the human protein, at least at the proprietorial arginine residues. These results prompted us to perform a comprehensive substrate screening of PRMT1 using a $\lambda \mathrm{gt} 11-$ HeLa cDNA expression library in E. coli host with IPTG-induced cDNA protein products separated on the nitrocellulose membrane.

### 3.2. Expression screening of PRMT1 substrates

Fig. 2 shows the experimental order of the screening (Fig. 2A) and a representative of the first and third screening (Fig. 2B). After the first screening, 607 positive clones were detected in $2.0 \times 10^{5}$ plaques. We picked up 100 of these clones and succeeded to isolate 83 positive clones by the third screening. We determined the nucleotide sequences of 48 of these clones. A BLAST search [17] against GeneBank revealed that more than half of the clones encode putative RNA-binding proteins such as hnRNP R, hnRNP K, GRY-RBP (glycine, arginine, and tyrosine-rich RNA binding protein), TAFII 68 (TBP associated factor $\mathrm{II}_{68}$ ), RBP56 (RNA binding protein 56), and TLS/FUS (translocated in liposarcoma), as summarized in Table 1. A common feature of these proteins is to contain RGG motifs at their C-terminus (indicated by stripes in the right end column of Table 1). Eight clones encoded the same novel protein, which also has RGG motifs at its C-terminus, but not other RNA-binding motifs. We termed this clone SAMT1 (substrate of arginine methyltransferase 1). The predicted amino acid sequence of SAMT1 is shown in Fig. 3A. During preparation of this manuscript, a clone named CG1-55 coding the identical protein to SAMT1 was registered to the database of GeneBank, as one of conserved human genes by comparative proteomics [18]. Northern blot analysis of SAMT1 mRNA showed ubiquitous expression in human tissue samples (Fig. 3B). We tagged cloned cDNAs with green fluorescent protein (GFP) and GFP-tagged SAMT1 was predominantly localized to the cytoplasm around nucleus, presumably at the endoplasmic reticulum (ER), while the other RGG-proteins were localized in the nucleus in HeLa cells (Fig. 4). Other five clones encode a transcription factor, ZF5 (zinc finger 5) and a GPI (glycosyl phosphatidylinositol)-anchor protein, p137GPI. Both of these proteins do not have typical RGG motifs.

### 3.3. Methylation of cloned proteins by PRMT1 in vitro

To determine whether the cloned proteins are really methylated by PRMT1 in vitro, we prepared recombinant proteins of GST (glutathione sulfatransferase)-ZF5, GSTp137GPI, GST-hnRNP R, GST-GRY-RBP, GST-TAFII 68 , GST-TLS, and GST-SAMT1 (Fig. 5A). These proteins were then incubated at $37{ }^{\circ} \mathrm{C}$ for $30 \mathrm{~min}$ with recombinant PRMT1 and ${ }^{3} \mathrm{H}$-SAM. All of above proteins were highly methylated by PRMT1, while GST alone was not methylated (Fig. 5A). These proteins were methylated when they were incubated with HeLa cell extracts and ${ }^{3} \mathrm{H}$ SAM (Fig. 5C). Next, we tested the effect of a dominantnegative PRMT1 on methylation of HeLa cell extracts. We mutated PRMT1 cDNA by substituting alanines for three amino acids, valine 63 , leucine 64 , and aspartic acid 65 , located in the highly conserved region that is proposed to be important for SAM binding and thus for methytlransferase activity [10] (Fig. 5B). The VLD to AAA mutation of PRMT1 abolished the enzymatic ability to methylate hnRNP A1 in vitro (data not shown). Administration of the mutant PRMT1 suppressed the methylation of GST-hnRNP A1,GST-TAFII $_{68}$, and GST-SAMT1 by the extract (Fig. 5C), suggesting that these proteins are methylated mainly by PRMT1 in the HeLa cell extract.

![](https://cdn.mathpix.com/cropped/2024_04_28_1c2a0b05acc9b6814cc7g-06.jpg?height=810&width=1186&top_left_y=1578&top_left_x=451)

Fig. 4. Intracellular localization of cloned proteins. HeLa cells were transfected with the expression vectors, fixed, and analyzed by indirect immunofluorescence staining. PRMT1 was expressed as a fusion protein with the HA epitope and stained with an anti-HA monoclonal antibody (a). The cloned RGG proteins, hnRNP R (b), GRY-RBP (c), TLS (d), TAFII 68 (e), and SAMT1 (f) were tagged with GFP protein and stained with anti-GFP monoclonal antibody. Fluorescein isothiocyanate-labeled anti-mouse second antibody was used to visualize each protein. Scale bar in (a): $20 \mu \mathrm{m}$.

(A)

![](https://cdn.mathpix.com/cropped/2024_04_28_1c2a0b05acc9b6814cc7g-07.jpg?height=1319&width=1063&top_left_y=280&top_left_x=486)

Fig. 5. Methylation of cloned proteins by PRMT1 in vitro. (A) Recombinant proteins of GST-hnRNP R (lane 1), GST-GRY-RBP (lane 2), GST-TAFII (lane 3), $^{2}$, GST-TLS (lane 4), GST-SAMT (lane 5), GST-ZF5 (lane 6), and GST-GPI anchor protein (lane 7) were purified from E. coli and incubated with [methyl $\left.{ }^{3} \mathrm{H}\right]$-SAM in the presence of recombinant PRMT1. GST was used as a negative control (lane 8). Methylation of each protein was detected by autoradiography (upper panels) and protein amount of each methylation reaction was confirmed with Coomassie staining (arrow heads of lower panels). (B) The ${ }_{63}$ VLD $_{65}$ to AAA mutation of PRMT1. (C) GST-hnRNP A1, GST-TAFII 68 and GST-SAMT1 were incubated with HeLa cell extract, ${ }^{3} \mathrm{H}$-SAM, and indicated amount of PRMT1 VLD to AAA mutant at $30^{\circ} \mathrm{C}$ for $1 \mathrm{~h}$. The amount of the RGG proteins were normalized by Coomassie staining as shown in the lower panels.

## 4. Discussion

Protein function is often modulated by posttranslational modifications; phosphorylation, acetylation, glycosylation, and methylation. Though an important role for protein methylation in bacterial chemotaxis has long been appreciated (reviewed in Ref. [19]), a role of protein methylation in signal transduction has not been explored in mammalian cells with the same breadth and intensity that protein phosphorylation has enjoyed in recent years. Unlike phos-phorylation, protein methylation might be irreversible, because protein demethylases have not been discovered so far [20]. Yet, this may be a key mechanism of cell function as a growing number of proteins are being found to contain methylated amino acids, monomethylarginine and asymmetric dimethylarginine. Many of these proteins are associated with RNA, such as the heterogeneous nuclear ribonucleoprotein (hnRNP) A1 [21,22], the U3 small nuclear RNAbinding protein fibrillarin [23], the rRNA binding protein nucleolin [24], poly(A)-binding protein (PAB) II [25], the Src-associated substrate, Sam68 [26], interleukin enhancer binding factor3 [27], and the yeast Npl3 protein [28,29]. As a result, the hnRNPs contain about $65 \%$ of the total $N^{\mathrm{G}}, N^{\mathrm{G}}-$ dimethylarginine found in the HeLa cell nucleus [11].

The site of arginine methylation for type I enzymes typically occurs within glycine- and arginine-rich (GAR) domains from which similar consensus sequences have been derived $[4,7,16,30]$. A search of the GenPept database for a consensus FGGRGGF motif revealed 13 substrate candidates [16]. Of these, 10 were known or presumed to interact with RNA [16]. In our expression screening, we did not isolate any of 13 candidates, and seven of cloned 9 genes encode hnRNP R, hnRNP K, GRY-RBP, TAFII 68 , RBP58, TLS/FUS, and SAMT1, respectively, all of which contain the consensus sequence or consensus-like sequences with two to three amino acid substitutions (Table 2). However, the other two do not have the consensus motif. Recently, Smith et al. [25] reported that most of the methylated arginine residues were found in Arg-X amino acid-Arg (RXR) clusters in the C-terminus of PAB2. They showed both PRMT1 and PRMT3 methylated the arginine in the Cterminal domain. As shown in Table 2, ZF5 and p137GPI contained the RXR sequence, though the methylation sites of these proteins have not been identified yet.

The glycine-rich clusters with many glycine-glycine dipeptides interspersed with aromatic or arginine residues are termed "RGG" [31]. The RGG motif is ubiquitous in proteins involved in diverse aspects of RNA metabolism such as pre-rRNA processing factors, pre-mRNA splicing factors, hnRNPs, and RNA helicases [32]. The C-terminal RGG motif of hnRNP U, which lacks RRM (RNA recognition motif), facilitate as the RNA-binding motif [31]. In nucleolin, which has two RRMs, the C-terminal RGG domain mediates RNA binding even without the RRMs [31]. However, Bouvet et al. [33] reported that this domain interacts with 10 ribosomal proteins. Likewise, in hnRNP A1, RGG domain is not only involved in interaction with RNA [34] but also with proteins [35]. In spite of prevalence of RNA binding proteins with RGG motifs, few studies have directly addressed the effect of arginine methylation on binding of target proteins to RNA. However, recent studies have examined arginine methylation in modulating protein-protein interactions (reviewed in Ref. [36]). Asymmetric arginine methylation of Sam68 within the RGG domains decreases Sam68 binding to SH3 proteins but not to WW domain-containing proteins, suggesting that methylation may alter the specific protein-protein contacts [37]. Similarly, methylation of STAT1 (signal transducer and activator of transcription) decreases its affinity for its inhibitor PIAS1 [38]. Recently, we screened the factors that interact with the RGG domain of Npl3p with the two hybrid system and isolated nine different genes of which five have RGG domains [39].

Table 2

(F/G)GGRGG(F/G) homologous sequences and RXR sequences found in cloned proteins

<table>
  <thead>
    <tr>
      <th>Gene</th>
      <th>(F/G)GGRGG(F/G) homologous sequence</th>
      <th>RXR sequence</th>
      <th>Gene</th>
      <th>(F/G)GGRGG(F/G) homologous sequence</th>
      <th>RXR sequence</th>
    </tr>
    <tr>
      <td rowspan="5">hnRNP R</td>
    </tr>
    <tr>
      <th>447aa-GRGRGGGRGGY-458aa</th>
      <th>106aa-RQR-108aa</th>
      <th>RBP58</th>
      <th>203aa-GGDRGGF-209aa</th>
      <th>185aa-RGR-187aa</th>
    </tr>
    <tr>
      <th>500aa-RGRGGGRGGRG-509aa</th>
      <th>306aa-RRR-308aa</th>
      <th></th>
      <th> 334aa-RRGRGGYRGRGGFQ-</th>
      <th>395aa-RGR-397aa</th>
    </tr>
    <tr>
      <th>540aa-RGSRGGRGGP-549aa</th>
      <th> 516aa-RGR-518aa</th>
      <th></th>
      <th> GRGGD-352aa</th>
      <th></th>
    </tr>
    <tr>
      <th></th>
      <th>554aa-RGR-556aa</th>
      <th></th>
      <th>456aa-GGDRGGG-462aa</th>
      <th></th>
    </tr>
    <tr>
      <td rowspan="6">GRY-RBP</td>
    </tr>
    <tr>
      <th>445aa-GRGRGGRGGY-455aa</th>
      <th> 103aa-RQR-105aa</th>
      <th></th>
      <th> 464aa-GGDRGGG-470aa</th>
      <th></th>
    </tr>
    <tr>
      <th>495aa-ARGRGGR-501aa</th>
      <th>303aa-RRR-305aa</th>
      <th></th>
      <th>472aa-GGDRGGG-478aa</th>
      <th></th>
    </tr>
    <tr>
      <th>536aa-RGARGGA-541aa</th>
      <th> 510aa-RGR-512aa</th>
      <th></th>
      <th>480aa-GGDRGGY-486aa</th>
      <th></th>
    </tr>
    <tr>
      <th>551aa-RGARGGRGGN-560aa</th>
      <th> 518aa-RGR-520aa</th>
      <th></th>
      <th> 487aa-GGDRGGG-493aa</th>
      <th></th>
    </tr>
    <tr>
      <th></th>
      <th>556aa-RGR-558aa</th>
      <th></th>
      <th>495aa-GGDRGGY-501aa</th>
      <th></th>
    </tr>
    <tr>
      <td rowspan="3">hnRNP K</td>
    </tr>
    <tr>
      <th>255aa-MRGRGGF-261aa</th>
      <th>35aa-RSR-37aa</th>
      <th></th>
      <th>502aa-GGDRGGY-508aa</th>
      <th></th>
    </tr>
    <tr>
      <th>296aa-RGGRGGS-302aa</th>
      <th>326aa-RGR-328aa</th>
      <th></th>
      <th>509aa-GGDRGGY-515aa</th>
      <th></th>
    </tr>
    <tr>
      <td rowspan="17">TAFII 68</td>
    </tr>
    <tr>
      <th> 200aa-GGDRGGF-206aa</th>
      <th> 182aa-RGR-184aa</th>
      <th></th>
      <th> 516aa-GGDRGGY-522aa</th>
      <th></th>
    </tr>
    <tr>
      <th>331aa-RRGRGGYRGRGGF-343aa</th>
      <th> 392aa-RGR-394aa</th>
      <th></th>
      <th> 532aa-GGDRGGG-538aa</th>
      <th></th>
    </tr>
    <tr>
      <th>400aa-GYRGRGGRGGDRG-</th>
      <th> 523aa-RSR-525aa</th>
      <th></th>
      <th> 559aa-GGDRGGG-565aa</th>
      <th></th>
    </tr>
    <tr>
      <th>GYGG-417aa</th>
      <th></th>
      <th></th>
      <th>567aa-GGDRGGY-573aa</th>
      <th></th>
    </tr>
    <tr>
      <th>453aa-GGDRGGG-459aa</th>
      <th></th>
      <th>TLS/FUS</th>
      <th>241aa-PRGRGGGRGGRGGM-254aa</th>
      <th>216aa-RGR-218aa</th>
    </tr>
    <tr>
      <th>469aa-GGDRGGG-475aa</th>
      <th></th>
      <th></th>
      <th>404aa-GGGRGGF-410aa</th>
      <th></th>
    </tr>
    <tr>
      <th>467aa-GGDRGGY-483aa</th>
      <th></th>
      <th></th>
      <th>473aa-RGGRGGY-479aa</th>
      <th></th>
    </tr>
    <tr>
      <th>484aa-GGDRGGG-490aa</th>
      <th></th>
      <th></th>
      <th>484aa-YRGRGGDRGGFRGGRGG-</th>
      <th></th>
    </tr>
    <tr>
      <th>492aa-GGDRGGY-498aa</th>
      <th></th>
      <th></th>
      <th>GDRGGF-506aa</th>
      <th></th>
   </tr>
    <tr>
      <th>499aa-GGDRGGY-505aa</th>
      <th></th>
      <th> SAMT1/CGI-55</th>
      <th> 162aa-IRGRGGL-168aa</th>
      <th> 134aa-RER-136aa</th>
   </tr>
    <tr>
      <th>506aa-GGDRGGY-512aa</th>
      <th></th>
      <th></th>
      <th>169aa-GRGRGGR-175aa</th>
      <th></th>
   </tr>
    <tr>
      <th>513aa-GGDRGGY-519aa</th>
      <th></th>
      <th></th>
      <th> 364aa-RPGRGGRGGRGGRGR-</th>
      <th></th>
   </tr>
    <tr>
      <th>556aa-GGDRGGG-562aa</th>
      <th></th>
      <th> Zinc finger 5</th>
      <th> ( - )</th>
      <th> 291aa-RLR-293aa</th>
    </tr>
    <tr>
      <th>564aa-GGDRGGY-570aa</th>
      <th></th>
      <th></th>
      <th></th>
      <th>382aa-RHR-384aa</th>
    </tr>
    <tr>
      <th></th>
      <th></th>
      <th>p137GPI</th>
      <th>( - )</th>
      <th>303aa-RQR-305aa</th>
  </tr>
</table>

Disruption of $\mathrm{Hmtlp} / \mathrm{Rmtlp}$ in yeast showed that the methyltransferase activity is critical for efficient nuclear transport of the specific hnRNP proteins, Npl3p, and Hrp1p, and is important in RNA metabolism [40]. Npl3p is also modulated by phosphorylation in the arginine/serine (RS) domain by Skylp and the phosphorylation influences the shuttling of Npl3p [41,42], indicating the complex regulation of hnRNP proteins with double modulation of arginine methylation and serine phosphorylation. In mammals, mouse embryonic stem cells bearing a null mutation of PRMT1 were viable, though the mouse was embryonic lethal [15]. However, the yeast $\mathrm{Hmt} 1 \mathrm{p} / \mathrm{Rmtlp}$ deletion strain was viable under standard growth condition [40]. In support of this, transfected HeLa cells with the VLD to AAA mutant, the dominant-negative PRMT1, expression vector survived and showed normal growth (unpublished data). Desrosiers and Tanguay [43] and Najbauer et al. [44] reported that protein methylation is involved in cellular stress responses and the aging/repair of proteins. Considering that PRMT1 is the predominant type I arginine methylase in mammalian cells [9], PRMT1 may be required for specific developmental processes or for stress responses. Systematic screening of substrate proteins with the solid phase methylation reaction described here will contribute to identify new roles of PRMT family.

## Acknowledgements

Part of this work was performed in SEEDS laboratory of Yamanouchi Pharmaceutical Co., Ltd. for the Industrial Science and Technology Program supported by New Energy and Industrial Technology Development (NEDO). This work was partially supported by Research for the Future Program, the Inamori Foundation, Nissan Science Foundation, The Mitsubishi Foundation, and Grants from the Ministry of Education, Science and Culture in Japan. We thank H. Hattou, T. Urushido, and T. Ishiguro for technical supports and Dr. Erich D. Jarvis for editing and critical comments.

## References

[1] W.K. Paik, S. Kim, Studies on the stability of L-amino-acid oxidase of snake venom, Biochim. Biophys. Acta 139 (1967) 9-55.

[2] H.W. Lee, S. Kim, W.K. Paik, $S$-adenosylmethionine: protein-argi- nine methyltransferase: purification and mechanism of the enzyme, Biochemistry 16 (1977) 78-85.

[3] J.M. Aletta, T.R. Cimato, M.J. Ettinger, Protein methylation: a signal event in post-translational modification, Trends Biochem. Sci. 23 (1998) 89-91.

[4] J.D. Gary, S. Clarke, RNA and protein interactions modulated by protein arginine methylation, Prog. Nucleic Acid Res. Mol. Biol. 61 (1998) 65-131.

[5] G.S. Baldwin, P.R. Carnegie, Specific enzymic methylation of an arginine in the experimental allergic encephalomyelitis protein from human myelin, Science 171 (1971) 579-581.

[6] H. Brahms, J. Raymackers, A. Union, F. de Keyser, L. Meheus, R. Luhrmann, The C-terminal RG dipeptide repeats of the spliceosomal Sm proteins D1 and D3 contain symmetrical dimethylarginines, which form a major B-cell epitope for anti-Sm autoantibodies, J. Biol. Chem. 275 (2000) $17122-17129$.

[7] W.J. Lin, J.D. Gary, M.C. Yang, S. Clarke, H.R. Herschman, The mammalian immediate-early TIS21 protein and the leukemia-associated BTG1 protein interact with a protein-arginine $N$-methyltransferase, J. Biol. Chem. 271 (1996) 15034-15044.

[8] M.F. Henry, P.A. Silver, Novel methyltransferase (Hmtlp) modifies poly(A)+-RNA-binding proteins, Mol. Cell. Biol. 16 (1996) $3668-3678$.

[9] J. Tang, A. Frankel, R.J. Cook, S. Kim, W.K. Paik, K.R. Williams, S. Clarke, H.R. Herschman, PRMT1 is the predominant type I protein arginine methyltransferase in mammalian cells, J. Biol. Chem. 275 (2000) 7723-7730.

[10] D. Chen, H. Ma, H. Hong, S.S. Koh, S.-M. Huang, B.T. Schurter, D.W. Aswad, M.R. Stallcup, Regulation of transcription by a protein methyltransferase, Science 284 (2000) 2174-2177.

[11] Q. Liu, G. Dreyfuss, In vivo and in vitro arginine methylation of RNA-binding proteins, Mol. Cell. Biol. 15 (1995) 2800-2808.

[12] B.P. Pollack, S.V. Kotenko, W. He, L.S. Izotova, B.L. Barnoski, S. Pestka, The human homologue of the yeast proteins Skb1 and Hs17p interacts with Jak kinases and contains protein methyltransferase activity, J. Biol. Chem. 274 (1999) 31531-31542.

[13] H.S. Scott, S.E. Antonarakis, M.D. Lalioti, C. Rossier, P.A. Silver, M.F. Henry, Identification and characterization of two putative human arginine methyltransferases (HRMT1L1 and HRMT1L2), Genomics 48 (1998) 330-340.

[14] P. Zobel-Thropp, J.D. Gary, S. Clarke, $\delta-N$-methylarginine is a novel posttranslational modification of arginine residues in yeast proteins, $\mathrm{J}$. Biol. Chem. 273 (1998) 29283-29286.

[15] M.R. Pawlak, C.A. Scherer, J. Chen, M.J. Roshon, H.E. Ruley, Arginine $N$-methyltransferase 1 is required for early postimplantation mouse development, but cells deficient in the enzyme are viable, Mol. Cell. Biol. 20 (2000) 4859-4869.

[16] J. Najbauer, B.A. Johnson, A.L. Young, D.W. Aswad, Peptides with sequences similar to glycine, arginine-rich motifs in proteins interacting with RNA are efficiently recognized by methyltransferase(s) modifying arginine in numerous proteins, J. Biol. Chem. 268 (1993) $10501-$ 10509.

[17] S.F. Altschul, W. Gish, W. Miller, S.W. Myers, D.J. Lipman, Basic local alignment search tool, J. Mol. Biol. 215 (1990) 403-410.

[18] C.-H. Lai, C.-Y. Chou, L.-Y. Chang, C.-S. Liu, W.-C. Lin, Identification of novel human genes evolutionarily conserved in Caenorhabditis elegans by comparative proteomics, Genome Res. 10 (2000) 703-713.

[19] M.S. Springer, M.F. Goy, J. Adler, Protein methylation in behavioural control mechanisms and in signal transduction, Nature 280 (1979) $279-284$

[20] M.R. Stallcup, Role of protein methylation in chromatin remodeling and transcriptional regulation, Oncogene 20 (2001) 3014-3020.

[21] A. Kumar, K.R. Williams, W. Szer, Purification and domain structure of core hnRNP proteins A1 and A2 and their relationship to single-stranded DNA-binding proteins, J. Biol. Chem. 261 (1986) 11266-11273.

[22] S. Kim, B.M. Merrill, R. Rajpurohit, A. Kumar, K.L. Stone, V.V. Papov, J.M. Schneiders, W. Szer, S.H. Wilson, W.K. Paik, K.R. Williams, Identification of $\mathrm{N}(G)$-methylarginine residues in human heterogeneous RNP protein A1: Phe/Gly-Gly-Gly-Arg-Gly-Gly-Gly/Phe is a preferred recognition motif, Biochemistry 36 (1997) 5185-5192.

[23] M.A. Lischwe, R.G. Cook, Y.S. Ahn, L.C. Yeoman, H. Busch, Clustering of glycine and NG-NG-dimethylarginine in nucleolar protein C23, Biochemistry 24 (1985) 6025-6028.

[24] M.A. Lischwe, K.D. Roberts, L.C. Yeoman, H. Busch, Nucleolar specific acidic phosphoprotein C23 is highly methylated, J. Biol. Chem. 257 (1982) 14600-14602.

[25] J.J. Smith, K.P. Rucknagel, A. Schierhorn, J. Tang, A. Nemeth, M. Linder, H.R. Herschman, E. Wahle, Unusual sites of arginine methylation in Poly(A)-binding protein II and in vitro methylation by protein arginine methyltransferases PRMT1 and PRMT3, J. Biol. Chem. 274 (1999) 13229-13234.

[26] G. Wong, O. Muller, R. Clark, L. Conroy, M.F. Moran, P. Polakis, F. McCormick, Molecular cloning and nucleic acid binding properties of the GAP-associated tyrosine phosphoprotein p62, Cell 69 (1992) 551-558.

[27] J. Tang, P.N. Kao, H.R. Herschman, Protein-arginine methyltransferase I, the predominant protein-arginine methyltransferase in cells, interacts with and is regulated by interleukin enhancer-binding factor 3, J. Biol. Chem. 275 (2000) 19866-19876.

[28] M.S. Lee, M. Henry, P.A. Silver, A protein that shuttles between the nucleus and the cytoplasm is an important mediator of RNA export, Genes Dev. 10 (1996) 1233-1246.

[29] C.W. Siebel, C. Guthrie, The essential yeast RNA binding protein Np13p is methylated, Proc. Natl. Acad. Sci. U. S. A. 93 (1996) $13641-13646$.

[30] R. Rajpurohit, S.O. Lee, J.O. Park, W.K. Paik, S. Kim, Enzymatic methylation of recombinant heterogeneous nuclear RNP protein A1. Dual substrate specificity for $S$-adenosylmethionine:histone-arginine $N$-methyltransferase, J. Biol. Chem. 269 (1994) 1075-1082.

[31] M. Kiledjian, G. Dreyfuss, Primary structure and binding activity of the hnRNP U protein: binding RNA through RGG box, EMBO J. 11 (1992) 2655-2664.

[32] E. Birney, S. Kumar, A.R. Krainer, Analysis of the RNA-recognition motif and RS and RGG domains: conservation in metazoan premRNA splicing factors, Nucleic Acids Res. 21 (1993) 5803-5816.
[33] P. Bouvet, J.J. Diaz, K. Kindbeiter, J.J. Madjar, F. Amalric, Nucleolin interacts with several ribosomal proteins through its RGG domain, J. Biol. Chem. 273 (1998) 19025-19029.

[34] F. Cobianchi, R.L. Karpel, K.R. Williams, V. Notario, S.H. Wilson, Mammalian heterogeneous nuclear ribonucleoprotein complex protein A1: large-scale overproduction in Escherichia coli and cooperative binding to single-stranded nucleic acids, J. Biol. Chem. 263 (1988) 1063-1071.

[35] L. Cartegni, M. Maconi, E. Morandi, F. Cobianchi, S. Riva, G. Biamonti, hnRNP A1 selectively interacts through its Gly-rich domain with different RNA-binding proteins, J. Mol. Biol. 259 (1996) 337-348.

[36] A.E. McBride, P.A. Silver, State of the arg: protein methylation at arginine comes of age, Cell 106 (2001) 5-8.

[37] M.T. Bedford, A. Frankel, M.B. Yaffe, S. Clarke, P. Leder, S. Richard, Arginine methylation inhibits the binding of proline-rich ligands to Src homology 3, but not WW, domains, J. Biol. Chem. 275 (2000) $16030-16036$.

[38] K.A. Mowen, J. Tang, W. Zhu, B.T. Schurter, K. Shuai, H.R., Herschman, M. David, Arginine methylation of STAT1 modulates IFNalpha/ beta-induced transcription, Cell 104 (2001) 731-741.

[39] K. Inoue, T. Mizuno, K. Wada, M. Hagiwara, Novel RING finger proteins, Airlp and Air2p, interact with Hmtlp and inhibit the arginine methylation of Np13p, J. Biol. Chem. 275 (2000) 32793-32799.

[40] E.C. Shen, M.F. Henry, V.H. Weiss, S.R. Valentini, P.A. Silver, M.S. Lee, Genes Dev. 12 (1998) 679-691.

[41] C.Y. Yun, X.D. Fu, Conserved SR protein kinase functions in nuclear import and its action is counteracted by arginine methylation in Saccharomyces cerevisiae, J. Cell Biol. 150 (2000) 707-718.

[42] W. Gilbert, C.W. Siebel, C. Guthrie, Phosphorylation by Skylp promotes Npl3p shuttling and mRNA dissociation, RNA 7 (2001) 302-313.

[43] R. Desrosiers, R.M. Tanguay, Methylation of Drosophila histones at proline, lysine, and arginine residues during heat shock, J. Biol. Chem. 263 (1988) 4686-4692.

[44] J. Najbauer, J. Orpiszewski, D.W. Aswad, Molecular aging of tubulin: accumulation of isoaspartyl sites in vitro and in vivo, Biochemistry 35 (1996) 5183-5190.